Hai secret no more (#4952)
endless-sky/endless-sky: data/hai/hai reveal 1 intro.txt

		"attitude toward"
			"Deep Security" 1.0

event "hai ladybug available"
	shipyard "Hai Intermediate"
		"Ladybug"
	shipyard "Hai Advanced"
		"Ladybug"

mission "Hai Reveal: Reputation Reset"
	landing
	invisible
	to offer
		has "event: pug media coverage"
	on offer
		# Sync reputations for aliases of governments:
		"reputation: Deep Security that won't enter wormhole" = "reputation: Deep Security"
		"reputation: Republic that won't enter wormhole" = "reputation: Republic"
		"reputation: Free Worlds that won't enter wormhole" = "reputation: Free Worlds"
		"reputation: Syndicate that won't enter wormhole" = "reputation: Syndicate"

		# Make sure the new governments start with appropriate rep:
		"reputation: Human Alliance that won't enter wormhole" = 1000
		"reputation: Human Alliance" = 1000
		"reputation: Hai (Friendly Unfettered)" = 1000
		"reputation: Bounty Hunter that Attacks Hai" = -1000
		"reputation: Philosophers" = 1000
		"reputation: Pirate (Devil-Run Gang)" = -1000
		fail

mission "Event: Pug Media Coverage"
	landing
	source
		government "Republic" "Syndicate" "Free Worlds"
		attributes "spaceport"
	to offer
		has "event: pug media coverage"
	on offer

> 		event "hai secret leaks" 10 15

Reviewer: Given that some of the dialog seems contingent on the main campaign having occurred recently, but an old pilot updating to a version that includes this content could have spent years past the main campaign, might we want to move the trigger for all this to the final mission of the main campaign instead of an invisible mission that triggers if the main campaign is done? If this is merged alongside a rework of the FW campaign, I imagine it could easily be justifiable to lock all this to new pilots.

> 		event "hai secret leaks" 10 15

Author: IMO the events really only make sense as an immediate follow on. If we extended the trigger it would be purely for gameplay reasons, not anything else.
I'm also inclined to say that old pilots *expect* new content might have potentially weird time-gaps. That's just a feature of games updating. I thought the whole point was for ES to be backwards compatible, so people didn't have to start new games to experience new content that wouldn't normally have been locked out? This suggestion seems to counter that philosophy.

> 		event "hai secret leaks" 10 15

Reviewer: We've had lockouts on new content in the past, but those were typically as a result of minor additions to existing campaigns as opposed to entire new storylines. The biggest lockout that comes to mind is the second half of the Terraforming missions, where players who had completed Terraforming Rand before Terraforming Tundra couldn't do Terraforming Tundra without a new pilot (or save editing, of course), and the reason for that was explicitly because of the lead-in to Terraforming Tundra being time-sensitive relative to when you completed Terraforming Rand.

And I'd say the events only make sense as an immediate follow up insofar as that's how you've written it. I don't see why the intro couldn't be rewritten to occur some time later. And the gameplay reasons for why we would extend the trigger time is entirely what my concern is. As of right now once you finish the main campaign the world is opened up to you, and there are already a number of existing missions that can only be done after the main campaign is complete. By having these events occur so quickly after the main campaign while also having no way to story wise break out of them (no defers or declines as far as I've seen), you suffocate out those other missions, namely the FW epilogues (not too important, although I imagine those epilogues would change when you meet people who you met during this storyline) and Emerald Sword missions. In the latter case it seems to especially be an issue to me that the events of Hai Reveal lead you right to the Wanderers (as far as I've seen from reading your comments elsewhere), which I'd say even further devalues the Emerald Sword by opening up the player to T2 tech before giving them a real chance to get their hands on the Emerald Sword. And should there be any future post-campaign content that's still contained within human space, this may crowd those out too.

I also take issue with this leading to the Wanderers in that it takes away from the exploration aspect of the game by pointing you to where you "need" to go. (I put need in quotes because technically a player could stall completing any mission at any time and go explore elsewhere, but simply having a mission pressures the player into doing that mission, thereby creating a perceived need to do things a certain way.) As mentioned, after beating the main campaign the game is opened up to you and you can go where ever you want with your new found jump drive, but these missions, in being forced (i.e. can't be deferred or declined) and so immediate after the main campaign, create a continuous pipeline from starting the main campaign up to beating it, starting Hai reveal and beating it, then starting the Wanderer campaign, leaving little room to explore elsewhere, such as exploring the Coalition or Remnant, and basically handing you the Wanderer campaign instead of having it be something you stumble upon as is done now.

> 		event "hai secret leaks" 10 15

Author: I understand your concerns, though I do have a few points:

> I don't see why the intro couldn't be rewritten to occur some time later.

I don't see *how* it *could* be rewritten to occur some time later. It takes advantage of very specific conditions that would only realistically be in place for a short time.

> you suffocate out those other missions, namely the FW epilogues

To be honest, those could (and some of them possibly should) just be moved to after this.

> and Emerald Sword missions

I don't recall those actually being all that easy to trip over tbh, and the Emerald Sword is a spectacular unique flagship one way or another. To me this seems like a non-issue.

> In the latter case it seems to especially be an issue to me that the events of Hai Reveal lead you right to the Wanderers
etc.

Eh, sort of? But also not really. It hints that the Unfettered have something going on with another race, and then once it's all over and done with, if you don't go to the wanderers for a decent length of time, there's a "You know you saw something that pointed to this section of space" type nudge. There's also that a lot of the community consider the Wanderers to be "the main plot part 2" and at present it's *very* disconnected, and lets you start it kind of illogically before the main campaign, but then hangs up in the middle without explanation until you do the main campaign. As far as the Wanderer's plot is concerned all this does is provide some more structure to the narratives leading up to and surrounding it.

> which I'd say even further devalues the Emerald Sword by opening up the player to T2 tech before giving them a real chance to get their hands on the Emerald Sword

And again, the Emerald sword isn't valued for what Tier it is, it's valued entirely independent of that.

		conversation
			`As you walk through the spaceport, you can't help but notice that coverage on the events of the Pug invasion still continues to dominate headlines, even though it's been weeks now.`
			`	Looking around, you see that experts and officials have given way to talking heads and theorists. Some of them are saying what you yourself suspect: that the Pug invasion was a ploy to cause humanity to band together, learn to resolve differences without conflict, and ensure the sorts of extremism that lead to the events precipitating the war would not be tolerated.`
			`	Others...`
			`	You sigh deeply as you watch some paranoid fringe world politician demand that the edges of human space be better understood, that threats and even potential threats must clearly exist beyond what is already known, and that it is the responsibility of the government to find out and inform the people.`
			`	Your thoughts, you keep to yourself.`
				decline

event "hai secret leaks"

> conversation "hai reveal free worlds news"

Author: I recall you mentioning in the past that you liked there to be examples of different functionality presented occasionally. I think this is a particularly good example of named conversations being shared.

But also on reflection I recall another reason for doing it this way:
When other campaigns get added there will be even more possible combinations. As such the objective here is to matrix it out, so that all geographic starting points are defined by the missions - and then the conversations handle the combination of branches required for each campaign in a given geographic area.

This makes it clearer and easier to work on and expand in the future when required.

As such I would like to maintain this structure.

> 	`You are relaxing at the spaceport while your ship refuels after conducting a spot of business. On the view-screen is some talk show - not something you specifically pay attention to, but apparently a quite popular one out of Tarazed.`

Author: I think the "a" is still necessary? Maybe it needs to go after "quite" instead of before.

> 	`You are relaxing at the spaceport while your ship refuels after conducting a spot of business. On the view-screen is some talk show - not something you specifically pay attention to, but apparently a quite popular one out of Tarazed.`

Reviewer 2: Yep. After "quite."

> 	`You are relaxing at the spaceport while your ship refuels after conducting a spot of business. On the view-screen is some talk show - not something you specifically pay attention to, but apparently a quite popular one out of Tarazed.`

Author: ```suggestion
	`You are relaxing within the spaceport while your ship refuels after conducting a spot of business. On the view-screen is some talk show - not something you specifically pay attention to, but apparently quite a popular one out of Tarazed.`
```

	`	A young and fresh-faced man who looks incredibly ill at ease has just been brought out as a surprise guest by the host Jonathon Camembert.`

> 	`	"So now in our surprise segment, let me introduce you to Ronald. He once crewed on one of these big trade ships, but after seeing what he saw, he just had to come forward!"`

Author: ```suggestion
	`	"So now in our surprise segment, let me introduce you to Ronald Campbell. He once crewed on one of these big trade ships, but after seeing what he saw, he just had to come forward!"`
```

> 	`	"So now in our surprise segment, let me introduce you to Ronald. He once crewed on one of these big trade ships, but after seeing what he saw, he just had to come forward!"`

Reviewer: Mmm. Soup.

	`	The host stresses the word 'had' so much his voice squeaks, but the audience's attention has certainly been gained.`
	`	"But first, let's take a look at a clip."`
	`	You raise an eyebrow with surprise.`
	branch "hai knowledge"
		has "First Contact: Hai: offered"
	`	A clip of alien space ships in combat with weapons quite unlike anything produced in human space is shown. You've seen things, but not this.`

> 	`	When it's finished the young man, who is very well presented but clearly has a bit of the paranoid about him that is plain to see even through a screen, is encouraged to talk about where he got the footage. He talks at length about an ongoing conflict that threatens to spill over and drag all of humanity back to war: a war waged by an alien species known as the 'Hai'.`

Reviewer 3: I'm actually a fan of the `paranoia` change here and on line 114. What was the specific reason against it, @MasterOfGrey ?

> 	`	When it's finished the young man, who is very well presented but clearly has a bit of the paranoid about him that is plain to see even through a screen, is encouraged to talk about where he got the footage. He talks at length about an ongoing conflict that threatens to spill over and drag all of humanity back to war: a war waged by an alien species known as the 'Hai'.`

Author: It's kinda the difference between a simile and a metaphor.

To say he has "a bit of the paranoid" about him is to say that he presents with mannerisms that are like the behaviour of those with the heightened anxiety of paranoia.
To say he has "a bit of paranoia" about him implies that he's actively saying things which indicate paranoia.

To give an example: In the first case one might look over one's shoulder before saying something that is risky or dangerous. In the second case one would explicitly refer to the expectation of someone coming to get them for what they're saying/about to say.

Apologies for not getting to that explanation sooner.

> 	`	When it's finished the young man, who is very well presented but clearly has a bit of the paranoid about him that is plain to see even through a screen, is encouraged to talk about where he got the footage. He talks at length about an ongoing conflict that threatens to spill over and drag all of humanity back to war: a war waged by an alien species known as the 'Hai'.`

Author: Alternative?
```suggestion
	`	When it's finished the young man, who is very well presented but is clearly anxious about being here, is encouraged to talk about where he got the footage. As he speaks he has a bit of a paranoid manner to him that is plain to see even through a screen, talking at length about an ongoing conflict that threatens to spill over and drag all of humanity back to war: a war waged by an alien species known as the 'Hai'.`
```

	label main
	`	"Unfortunately, it's taking a while for them to redistribute an appropriately resourced security force thanks to the Unfettered."`
	`	Your conversation is interrupted suddenly by a small group of somewhat official-looking Hai.`
	`	"Are you Captain <last>?" one of them asks you. "Don't worry, you're not in trouble or anything."`
	`	You nod your affirmation.`
	`	"Your reputation precedes you, Captain. If you have any stake in the current information crisis, please head directly to Hai-home. Someone would very much like to talk to you. As a person of note, you are presently being treated as an Envoy." They leave without waiting for your agreement. Then again, you did come here to see what was going on.`
		decline

# Six possibilities for origin and destination, to work around bug 1938
# Symbols RS=Republic or Syndicate FW = Free Worlds
# H=met Hai, U=met Unfettered

> mission "ZZ Event Hai Secret Leaks: RSHU"

Reviewer: I spotted later that the load order is necessary for how the tour is done, but these "Secret Leaks" missions don't need to manipulate any load orders.

> mission "ZZ Event Hai Secret Leaks: RSHU"

Author: On reflection, if I recall correctly this was done so that if any other missions fire at the same time, then this (as a pseudo-campaign start) is the last thing the player has their attention on, so they don't get the start of this and then get distracted.

> mission "ZZ Event Hai Secret Leaks: RSHU"

Author: `	You suspect it might be time to go visit your Hai friends again.`
`	This is, of course, ridiculous... but maybe it would be good to go investigate this for yourself. You listen in to collect the relevant details for future reference.`

Not sufficient indicators of an objective?

> mission "ZZ Event Hai Secret Leaks: RSHU"

Author: Oh _that_ right, yes, I see. That I can deal with.

		not "First Contact: Hai: offered"
		not "asked to investigate hai leak"
	on offer
		event "pirates investigate leaks"
		event "news: hai video clip published"
		set "asked to investigate hai leak"
		conversation "hai reveal republic or syndicate news"

mission "ZZ Event Hai Secret Leaks: FW"
	landing
	name "Investigate the Hai"
	description "A secret and violent alien race waging war just outside of human space? Preposterous; go find out what's really going on."
	source
		government "Free Worlds"
		attributes "spaceport"
	destination "Sunracer"
	to offer
		has "event: hai secret leaks"
		not "First Contact: Hai: offered"
		not "asked to investigate hai leak"
	on offer
		set "asked to investigate hai leak"
		event "news: hai video clip published"
		event "pirates investigate leaks"
		conversation "hai reveal free worlds news"

event "pirates investigate leaks"
	system "Waypoint"
		government "Pirate"
		add fleet "Marauder fleet IX" 2000
		add fleet "Anti-Pirate Hai (No Wormhole Travel)" 700

mission "Trader Directions to Hai Space"
	landing
	name "Investigate the Hai"
	description "Follow the coordinates given to you by Captain Casey and meet the Hai."
	source Sunracer

> 	waypoint "Ultima Thule"

Author: I figure if you don't know of the Hai, then jump drive or not you don't really know where their space is - so if you get told a route, logically you'd have to follow it. Shame you can't hide the destination until you reach the waypoint?

	destination "Allhome"
	to offer
		or
			has "ZZ Event Hai Secret Leaks: RS: offered"
			has "ZZ Event Hai Secret Leaks: FW offered"
	on offer
		conversation

> 			`Arriving at the only place you can think of where you feel sure you can find out more information, you find that <origin>, like most places, is abuzz with the recent news. Avoiding the many large, cheap, and boisterous establishments that service the many crews and workers of the Syndicate corporate machinery here, you make your way to one of the marginally more up-market places, cunningly marketed as a merchant-only dive.`

Author: It's still a Syndicate affiliate.

I've always felt like Sunracer is the place where you'll find rumours and conspiracies actually spoken about with some degree of seriousness. It's always felt like the world to go to if you were looking for a hub where you'd find people that know a bit about the fringes of human space (at least in the north).

Idk if that needs to be made explicit or if that's an impression that's just me.

> 			`Arriving at the only place you can think of where you feel sure you can find out more information, you find that <origin>, like most places, is abuzz with the recent news. Avoiding the many large, cheap, and boisterous establishments that service the many crews and workers of the Syndicate corporate machinery here, you make your way to one of the marginally more up-market places, cunningly marketed as a merchant-only dive.`

Reviewer: > It's still a Syndicate affiliate.

The relationship between the Syndicate and Megaparsec isn't particularly detailed in game right now (although I plan to elaborate more on it in the Syndicate campagin). The most we have is that Megaparsec is on a world that the map marks as a Syndicate system, and Megaparsec supplies Syndicated Security with some of their ships. Outside of that though, the fact that it has its own name suggests that it's a separate entity, one that makes describing it as "Syndicate machinery" seem off to me.

> I've always felt like Sunracer is the place where you'll find rumours and conspiracies actually spoken about with some degree of seriousness.... Idk if that needs to be made explicit or if that's an impression that's just me.

Perhaps if there was more characterization to Sunracer outside of these missions suggesting such then I'd see where you're coming from, but as of right now I don't really get that feeling from Sunracer. If you asked me where you'd want to go in the galaxy to find out some seedy information, I'd suggest hitting up a pirate world. Now granted, Sunracer is on the edge of civilization and close to a pirate world, so perhaps a connection could be made from there, but such a connection should be made prior to us deciding that this is the best place to be to get the information we need. How that connection is made beforehand, though? No idea.

> 			`Arriving at the only place you can think of where you feel sure you can find out more information, you find that <origin>, like most places, is abuzz with the recent news. Avoiding the many large, cheap, and boisterous establishments that service the many crews and workers of the Syndicate corporate machinery here, you make your way to one of the marginally more up-market places, cunningly marketed as a merchant-only dive.`

Author: Aight, I can work from that.

> 			`	The endless cycle of both news and government propaganda displayed on most screens in major thoroughfares is remarkably absent here, with only one of the screens on display covering the "violent alien race" at all - running stats on how many different versions of the story are now being told.`

Author: I would expect "advertisements" to be what counted as government propaganda in Syndicate space, but fairo.

But I think in this case I might have actually been going for more of a "news and _'news'_ are crowding out advertisements and normal programming still at the moment" type deal.

			choice
				`	"Thank you."`
					goto wrap
				`	"The wormhole?"`
			`	She shakes her head. "See for yourself."`
			label wrap
			`	Taking a sip from her mug she nods slightly, once, in your direction. "Intentionally or not, you did me and a lot of other merchants a solid in that war, Captain. I respect that, but I'll thank you to leave me in peace if you see me again."`
			`	Only now, as you move to depart, do you spot the wrist-triggered needlegun strapped to the arm of her drinking hand that would've been trained on you the entire conversation.`
				accept
			

#This branch is if you have met the Hai but not the Unfettered
	#Goto Stop 3 Wah Yoot (Darkcloak) and pick up at Tour 4 mission.
mission "Hai Witness Unfettered [1]"
	name "Witness the Unfettered"
	description "It turns out this conflict has some truth to it. Go see where it's happening."
	landing
	source "Allhome"
	destination "Darkcloak"
	to offer
		or
			has "ZZ Event Hai Secret Leaks: RSH: done"
			has "ZZ Event Hai Secret Leaks: FWH: done"

# This fires if you haven't met the Hai, to trigger the entire tour.
# It must happen after the first contact mission.
	#Goto Stop 1 Imo Dep (Stonebreak)

> 	#This follows immediately after the First Contact text - no other convo is required.

Author: Mmmm, you are explicitly sent to Allhome to trigger this, but I see your point.

To what extent does this need to be fully idiot-proofed before merging though?
Both this and your point about restructuring the named conversations bit (which I thought was a good things as a demonstration of the functionality) feel like they'd be better stress-tested in their own PR after it's merged when we start seeing what people do to break it.

> 	#This follows immediately after the First Contact text - no other convo is required.

Reviewer: That whole "Merge things and fix them latter as people break them," is exactly something I've said I want to avoid. Let's get this right the first time. Breaking things is exactly what the review phase is for.

> To what extent does this need to be fully idiot-proofed before merging though?

I wouldn't call it idiot proofing as much as I would call it doing things as the game has done them already. There isn't anywhere that I can recall where the game has so strictly required you to go to exactly where you need to in order to have everything work properly, and if you land anywhere else you break thing. As I mentioned in another review comment somewhere (I think), the player has a jump drive at this point. It may be that they don't even enter Hai space through the wormhole and therefore Allhome wouldn't be the first planet that come across the land on.

> 	#This follows immediately after the First Contact text - no other convo is required.

Author: > I think an alternate first contact at this point is a perfectly reasonable expectation for this PR.

I think it would be perfectly reasonable if someone had got to this sooner than _two years_ after it opened. As it stands, no. It can happen after when there are fewer moving parts involved.

mission "ZZ Hai Tour [3] Meet Unfettered"
	landing
	name "Meet the Rebels on <planet>"
	description "Go to <destination> and learn more about the Hai's conflict."
	source "Alta Hai"
	destination "Darkcloak"
	to offer
		has "Hai Tour [2] Alta Hai: done"
	on offer
		conversation
			`This is a complete Quarg ringworld, the product of millennia of construction effort which has consumed every other stellar object in this system in order to provide the raw materials. The inner edge of the ring is filled with solar panels, hydroponic gardens, living quarters, and sunlit common spaces. The outer edge contains hangars and docks for ships while housing storage facilities for cargo.`
			`	Pondering your questions, you find a Quarg and ask, "So this isn't Hai space then, but it's a dead end?"`
			`	The Quarg explains, "Much like in Enif in human space, this is where we live as friends to the Hai."`
			`	This explanation seems typically Quarg, but it's the best one you're able to get. Before you go, you ask, "Do you know about the rebels I've been hearing about?"`
			`	The Quarg responds, "The Hai and their rebels are the same. You should travel to Darkcloak in Wah Yoot to understand."`

> 			`	It seems like there's nothing else for it but to see for yourself.`

Reviewer: This tour doesn't feel very organic and seems unnecessary to me. I'd say it could probably be removed and any information that it's suppose to provide to the player should be provided in later missions where necessary. If nothing else I'd say it needs heavily reworked to actually give something of value to the player outside of just saying "Oh check out this cool thing, not check out that cool thing," and you should use alternate versions of the first contacts where necessary instead of fudging the mission names to fiddle with the offer orders and having multiple missions stack on top of each other.

Author: The value it provides to the player (who: reminder, only gets this if they have never met the Hai) is that they fill out the map of all the Hai systems as prelude to needing to know where most of them are later on for plot reasons.

And also, this gives them a solid chance to simply look around and experience the Hai space before being thrust into the narrative.

Reviewer: Such goals can still be achieved without feeling so inorganic. As I said, "If nothing else I'd say it needs heavily reworked..."

				accept
	
	#Goto Stop 4 Wah Ki (Cloudfire)
	# Must happen before unfettered first contact
mission "00 Hai Tour [4] Cloudfire"
	landing
	name "Find out about the Rebels on Cloudfire"
	description "Go to Cloudfire and compare the Hai's side of the conflict."
	source "Darkcloak" 

> 	destination "Cloudfire"

Reviewer: Another mission where the destination is not made clear.

# Both tours merge here
mission "Hai Tour [5] Tour Ends"
	landing
	name "Head directly to Hai-home"
	description "Go to Hai-home and meet whoever wants to speak with you."
	source "Cloudfire"
	destination "Hai-home"
	clearance
	to offer
		has "00 Hai Tour [4] Cloudfire: done"
	on offer
		conversation
			branch "skipped tours"
				has "Hai Witness Unfettered [1]"
			`After looking around Cloudfire for some time, you stop in the spaceport for a moment to absorb everything you've seen. A screen is playing through news on the wall nearby, and your ears suddenly perk up at the word "marauder".`
			`	"Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew kidnapped nearly four dozen Hai and hijacked a Geocoris before making off with their spoils."`
				goto "that's awful"
			label "skipped tours"

> 			`Soon after you land on <planet>, you notice a news broadcast on a nearby wall. "Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew kidnapped nearly four dozen Hai and hijacked a Geocoris before making off with their spoils."`

Reviewer: Seems like this line could just be "Soon after you land on <planet>, you notice a news broadcast on a nearby wall." and the news report paragraph could be shared by both sides of the branch as a part of the "that's awful" label instead of needing to be duplicated.

			`	For a second you wonder if it's a joke, but they look as serious as Hai can look.`
			choice
				`	"I really don't think that's necessary."`
					goto nevertheless
				`	"Oh, I'm honored. Thank you."`
			`	"If you have any stake in the current information crisis, please head directly to Hai-home. Someone would very much like to talk to you."`
				goto main
			label nevertheless
			`	"You may not think it's necessary, but sometimes that is the price of fame." The official-looking Hai shrugs. "In any event, please head directly to Hai-home. Someone would very much like to talk to you."`
			label main
			`	They leave without waiting for your agreement, but then, you did come here to see what was going on.`
				accept

#All prior branches merge here.
	
mission "Hai Leaks Response 1"
	landing
	name "Hai message to <planet>"
	description "Take the diplomatic ID from the box from Sayari to <destination>."
	source "Hai-home"
	destination "Earth"
	to offer
		or
			has "Hai Tour [5] Tour Ends: done"
			has "ZZ Event Hai Secret Leaks: RSHU: done"
			has "ZZ Event Hai Secret Leaks: FWHU: done"
	on offer

> 		log "People" "Sayari" `Ambassador Sayari, a Hai, served as their envoy to the human government a century ago (although no public records of this exist in human space). Her experience has placed her in charge of coordinating the Hai response to the crisis of their continued secrecy.`

Reviewer: `action`s are newer, yes.

		log `The Hai are very real, and human worlds seem poised to become aware of their existence. The Hai would like to avoid this. Agreed to assist by representing their interests where travel to human space is required. Their desire was for a human, yet somewhat neutral, party.`
		conversation
			`As you arrive on Hai-home, you find that a reasonable security detachment is waiting at your designated landing berth. The Hai aren't usually ones to be concerned about such things, but you suspect that, as an apparent "war hero", they may be more for your own safety than the Hai's.`
			`	Word of your status in human space seems to have spread ahead of you, especially considering your apparent reputation on many Hai worlds as one of the few significant human visitors. A small crowd of Hai gather to peek at you on the short walk from the spaceport to the government building you are led to.`
			branch karma
				karma > 1

> 			`	As you are making the walk you hear questions posed and rumors stated. Occasionally you hear words like "survivor" and "dangerous" thrown in amongst the babble.`

Author: I do believe you have to have taken at least 1 fairly concerning decision in the FW plot for that to be your Karma though?

> 			`	As you are making the walk you hear questions posed and rumors stated. Occasionally you hear words like "survivor" and "dangerous" thrown in amongst the babble.`

Reviewer: Completing Wolf Pack (optional) drops your karma by 3.
FW Hope Recon 1C ups your karma by 1 if you let the Gunboat go, but you could have also just not even found the Gunboat.
Following the Senate's orders and going the diplomacy route for Thule ups your karma by 1.
FW Southern Prisoners gives +1 karma for releasing the prisoners and -1 karma for keeping them.
FW Bloodsea gives -1 karma if you attack Bloodsea.

So if you don't do Wolf Pack, don't find the Gunboat, follow the Senate's orders, don't attack Bloodsea, and then keep the prisoners, you end up with 1 karma, and I wouldn't say that keeping the prisoners is all that reprehensible an action as to get someone labeled as dangerous.

Karma is heavily underutilized though and there are plenty of decisions that should probably affect your karma in the FW campaign that don't, so perhaps we can just leave this as is and reassess it later along with the rest of the karma usage.

				goto main
			label karma
			`	As you are making the walk you hear questions posed and rumors stated. Occasionally you hear words like "honorable" and "reasonable" thrown in amongst the babble.`
			label main
			`	You arrive at a very comfortable office and are received by a mature-looking Hai with the appropriate regalia of a diplomat. She holds out a paw to greet you and says, "My name is Sayari. I was the envoy to the human government a century ago."`
			`	The present media storm is all the evidence you need that the Republic has never mentioned the Hai, nor publicly acknowledged receiving an envoy from them. If they're in on the secret, then they're probably still trying to work out a suppression strategy and have been too busy to send an envoy quicker than your own trip took.`
			`	Sayari continues, "Our reports tell us that you are a well-known and respected individual in human space. We also know 'Hai' is quickly becoming a buzzword in your space. A few traders brought the news ahead of you."`

> 			`	She pauses and looks at you seriously. "We think it would be prudent for you to represent your people as our point of contact. Would you accept such a role?"`

Author: At the moment, the assumption is fine - as mentioned *several* times, there will naturally be branches as other campaigns are added/when a FW checkmate rewrite actually eventuates.

			`	You give it a moment of thought.`
			choice

> 				`	"That seems practical."`

Author: For the moment, I figured a choice to decline the plot at certain points was a bit redundant. If the player really wants to bail they can always abort the mission.

			`	"Fantastic," she says, though her tone is much more business like than the wording would imply. "In that case, if you could please take this to Earth and communicate with your various governments we can begin a dialogue on how best to deal with this current scenario."`
			choice
				`	"Various? We only have two governments."`
			`	"Oh," she says, "we've treated the Syndicate as a de facto government for many decades now; with the amount of influence they have it was the only prudent thing to do."`

> 			`	You take the boxed item with you, and an escort awaits to return you to your ship. As the door behind you closes, you hear Sayari mutter to someone, "It really was much easier when they just had one government though. What a mess."`

Author: I think the best justification for this is not the original one; as became apparent on the recent Korath PR, Hai made contact with humanity by the timeline, immediately after dealing with the Korath raids and subsequently cleaning themselves up to look nice and professional - aggressively diving back into their philosophy of peace.

As far as they were concerned there would've been only one human government at the time, because that would have been a matter of officialdom, and also because pre-oracle Syndicate would have at least been balanced by the Paradise worlds. (Also the Deep is democratically and voluntarily a part of the republic that adopts Republic foreign policy and only seems to care about autonomy in internal affairs.) <- i.e. they were one government for all intents and purposes.

But since then... it's naïve to assume the Hai wouldn't know about the state of things in Human space - they'd just come out of a fresh conflict with the Korath, they'd absolutely put efforts into being aware of their neighbour's status'. Additionally, we know marauders and merchants both manage to get into Hai space at one time or another, and by simple geography, knowledge of the Syndicate's influence will be well-represented in those groups.

The Hai have twenty thousand years of accumulated cultural knowledge and tradition for every thousand years since we developed farming, and thirty thousand for every millennia since writing. With a hundred years of exposure and (as established later in this) good competency with holographics, I imagine the Hai government has a better understanding of the political breakdown in Human space than even some of our own elected officials.

				accept
	
	
mission "Hai Leaks Response 2"
	landing
	name "Collect Representatives"
	description "Collect human government representatives from <stopovers> and take them to <destination>."
	source "Earth"
	stopover "Deep"
	stopover "Foundry"
	destination "Hai-home"
	blocked "You have a mission here for the Hai, but you need three bunks free."
	clearance
	to offer
		has "Hai Leaks Response 1: done"
	passengers 3
	on offer
		event "pirates respond to leaks"
		log "People" "Xilin Yang" `Deputy Minister Yang works in the Foreign Affairs department of the Republic government. He is a pleasant and astute man. Handling the Hai secrecy crisis has become his job while the majority of the foreign ministry remains engaged with the fallout from the Free Worlds and recent peace.`
		conversation

> 			`On approach to the spaceport on Earth, you request to speak to an official of significant standing on diplomatic business. Thanks to your renown from the Pug war, you are able to shortcut objections stemming from you yourself not being a diplomat. The diplomatic ID from the box Sayari gave you doesn't hurt either.`

Author: ```suggestion
			`On approach to the spaceport on Earth, you request to speak to an official of significant standing on diplomatic business. Thanks to your renown from the Pug war, you are able to shortcut objections stemming from you yourself not being a diplomat. The diplomatic ID from the box Sayari gave you, when presented, also seems to trigger some sort of classified override fast-tracking the process from the quoted 'two to three business days' to '2 hours after landing'.`
```

			`	The Republic Deputy Minister for Foreign Affairs meets you in a private room at the spaceport.`
			`	"Captain <last>," he steps around the table to grasp your hand in a firm handshake. "Xilin Yang. It's an honor to meet a hero of the Pug crisis such as yourself."`
			choice
				`	"It was hard-fought, but it was necessary."`
				`	"Thanks, but it wasn't as dramatic as they made it out to be. I'm just happy to have helped where I could."`
					goto humble

> 			`	You play the part and try to act noble. Mr. Yang seems more than happy to embody respect for the hero of his preferred narrative.`

Author: This one happens to be the name of a colleague of mine. I thought it sounded cool and it's how he uses it so. idk.

			`	"You're every inch what I expected," he assures you before moving to take his seat.`
				goto sit
			label humble
			`	You shrug, palms up, and explain, "You should listen to the xenoanthropologists. The Pug were just here to test us, to make us really consider what mattered and come together."`
			`	For a moment his expression falters, and then he seems almost relieved. "You may be a better person than I expected to meet."`
			`	He offers you a genuine smile as he takes his seat.`
			label sit
			`	You sit down opposite him and explain how the Hai have proposed that you be their official contact point for humanity on account of your fame from the war.`
			`	He asks a few questions for clarity, and when you're done he requests you wait while he makes a call. He leaves the room for a long while, and at one point an aide comes in and offers you refreshments while deliberations continue elsewhere. Much to your surprise the fruity beverage is actually quite light and not too sweet, reminding you briefly that most of humanity's food styles originated here.`
			`	Eventually Mr. Yang returns. "So sorry for that. I hope you were looked after suitably while you waited."`

> 			`	Once you nod your assent, he continues, "Well, obviously the Republic knows about the Hai, but for the moment our official "line" is that the government does not trade in rumors. However, an appropriate response is clearly needed. Republic Intelligence briefed the Cabinet of the Free Worlds Senate just yesterday. They have demanded that they have a diplomatic representative along for any decisions. This is already standard practice for the Syndicate. If you are willing, I will come with you to meet two individuals known as Katya and Alastair on Deep and Foundry respectively. I believe you already know them?"`

Author: Referring to previous comment: `the Deep is democratically and voluntarily a part of the republic that adopts Republic foreign policy and only seems to care about autonomy in internal affairs.`

If this changes in other campaigns then it would naturally get branches here when they happen.

			choice
				`	"Yes."`
					goto conclude
				`	"Alastair?"`

> 			`	"Alastair Korban is the Syndicate CEO, naturally the CEO has some involvement here, but I trust that won't be a problem?"`

Author: I would love to include a better pick for the Syndicate, I just didn't want to name a person you'd then have to change or accommodate.
Also, in the immediate wake of recent events Katya's not an unreasonable pick - later on Katya just hands it over to Alondo so there is that.

> 			`	"Alastair Korban is the Syndicate CEO, naturally the CEO has some involvement here, but I trust that won't be a problem?"`

Reviewer 4: In "Paradise Fortune 3", Diana specifically calls to meet with Alondo, so Xilin calling for Katya is still kinda inexcusable.

> 			`	"Alastair Korban is the Syndicate CEO, naturally the CEO has some involvement here, but I trust that won't be a problem?"`

Author: Paradise Fortune doesn't necessarily occur so swiftly afterwards that the FW establishment hasn't really set up yet.
It's an indication that information propagation is still in progress post-war. Katya was well-known and easy to get a hold of, but as it is an Alondo job, she does hand it straight over.

> 			`	"Alastair Korban is the Syndicate CEO, naturally the CEO has some involvement here, but I trust that won't be a problem?"`

Reviewer 2: Cancel last. Saw below.

		conversation
			`You are directed to an official landing pad on your return to Hai-home, and it is not long before you are introducing your entourage to Sayari. Alondo in particular seems excited to be here and seems to have a knack for charming Sayari with polite and insightful, if rather simple, questions throughout the diplomatic niceties.`
			`	You get down to business when Sayari inquires how your journey went.`
			choice
				`	"I don't recall there being nearly so many pirates."`
			`	Sayari frowns and exhales a long breath through her nose, "Yes. Well, it seems that the rumors making their way through your media have attracted every opportunist this side of the Deep."`
			`	"Of course," she says, "some of the pirates were aware of us, and we were aware of them being aware of us. We had a quiet arrangement with a select few where we rewarded them for keeping the secret well contained within their own circles."`

> 			`	Alondo looks surprised, but Mr. Yang and Ms. Adrianopolous both merely nod.`

Author: This more a matter of it being not very kosher (from an international relations pov) to have independent agreements with dissident elements of a polity that you otherwise maintain an official relationship with.
This is an expression of surprise that the Hai treat with pirates as separate entities to the Republic, rather than treating with the Republic to handle them. The arrangement runs counter to conventional international relations policy and legitimises the independence of the pirates - something not usually accepted by government's like the Republic.

Alondo is from a place very far-removed from the Hai, this arrangement would be surprising. Especially as (by convention) it implies that between the Republic and the Hai, the Hai are the stronger of the two.

			`	He checks the date. "Our most optimistic estimates suggest the Republic will be forced to acknowledge the existence of the Hai within three months, or we would lose so much credibility as to compromise electoral stability. The actual time limit is probably sooner."`
			choice
				`	"Would that be so bad?"`
			`	"It is always bad to be forced into acknowledging a secret. If we act sooner, we may control the manner of that secret's release," says Yang. "Plus, while the Hai have let the trickle of humans that find the wormhole settle here peacefully, a massive influx is a problem nobody really wants to have to deal with."`
			`	Sayari looks at you. "We will have to come up with a plan, and quickly, but for the moment may I ask a favor of the war hero in the room? Please meet up with the security forces converging on Allhome and help them route the pirates back to the other side of the wormhole. It will buy us some breathing room and options."`

> 			`	It is so hard to say "no" to a Hai and their cute squirrelly features.`

Reviewer: If we want to use this to juxtapose against something later, maybe have this be said by a different character (Alondo?) instead of thought by the player. As with some other comments, this may be particularly off if the player has already met and interacted with the Hai before all this.

		conversation
			`When you land on Deep, Katya is there waiting for you along with Alondo. "<first>! It's so good to see you again! When the Republic briefed the new Chancellor and his Cabinet on the Hai, he brought us in for advice. We thought to try and reach you, but of course you would somehow manage to be in the thick of it before we even knew what was going on."`
			`	You hug them both in turn and return the greeting. "I thought it seemed odd when they said I was meeting you for this Katya, but it makes much more sense if you were just easier to get a hold of than Alondo here."`
			`	"Oh it was always going to be Alondo," she says, "but there's never a bad excuse for traveling with an old friend, especially when you're meeting other friends too."`
			`	"Do we have time for a drink before we leave?" asks Alondo.`
			`	You look around at the busy spaceport and see the workers haven't even noticed your ship waiting for refueling yet.`
			`	"I think we can spare a short while, and you really should meet Mr. Yang."`

> 			`	Mr. Yang had been studiously pretending he wasn't eavesdropping from a distance of a few meters but, before you could call him over, Alondo's slightly suggestive comment of, "Oh I should definitely meet Mr. Yang," had evidently been overheard.`

Author: I felt it made sense in the context of being immediately post-war, and with you as an intermediary. I would definitely have a branch here if it was coming off something other than the FW campaign which would work diferently..

				decline

mission "Hai Leaks Response 2C"
	landing
	invisible
	source Foundry
	to offer
		has "Hai Leaks Response 2: active"
	on offer
		log "People" "Terry Adrianopoulos" `Protege to the Syndicate CEO, Terry Adrianopoulos has been nominated by her mentor Alastair Korban to represent the Syndicate's interests in the crisis of the Hai's secrecy. She is a very capable woman who appears well equipped for climbing the corporate ladder.`
		conversation
			`Approaching Foundry, you are hailed on a private encrypted channel and directed to a private hangar where you are greeted personally by Syndicate CEO Alastair Korban.`
			`	He shakes your hand and smiles warmly. "Apologies, Captain <last>, do excuse the cloak and dagger, but in the current media climate you and I are both too well-known to be seen meeting without causing the rumor mill to grind out a new story."`
			`	"Now, allow me to introduce you to my protege, Terry Adrianopolous. She will be accompanying you as the Syndicate representative."`

> 			`	"Pleasure to meet you," she says with a broad smile. The young woman goes from calculating to gregarious in the blink of an eye as your attention shifts to her. She is pleasant and approachable, and very articulate, but from your brief glimpse beforehand you have no doubt she's cut from the sort of cloth that makes her future CEO material. You are surprised that a woman would have made it this far in the Syndicate, but if she has gotten this far, she must be very skilled at what she does.`

Reviewer: New tsundere just dropped.

			branch delayedretire
				not "free worlds reconciliation"
			`	He leans in close to you and says quietly, "I know I said I was retiring, but Terry convinced me to clean up my mess first. I am atoning for my failures. Do look after her; she's a very loyal colleague."`
			label delayedretire
			`	Alastair says goodbye and departs, leaving you with Terry.`
				decline

#This gets picked up silently on Hai-home when landing for response 2 - leads to go kill pirates mission with chat on Allhome.
mission "Hai Leaks Response 3"
	name "Meet with Security Forces on Allhome"
	description "Meet with the Security Forces on <destination> to fight pirates."
	landing
	source "Hai-home"
	destination "Allhome"
	to offer
		has "Hai Leaks Response 2: done"

#Go kill pirates mission with chat on Allhome.
mission "Hai Leaks Response 4"
	name "Defeat the Black Lich"
	description "Defeat Black Lich's Marauder Leviathan at <waypoints>, then return to <origin>."
	autosave
	landing
	source "Allhome"
	waypoint "Waypoint"

> 	destination "Allhome"

Reviewer: > It's necessary if you also have a waypoint.

You sure about that?

Author: Well, at an earlier stage in the process it didn't work without it. I do not recall the exact circumstances of that testing at the time - it being nearly 2 years ago.

			choice
				`	"I have heard about the kidnapping."`
			`	He nods. "Unfortunately, it's taking a while for them to redistribute an appropriately resourced defense force, thanks to the Unfettered. It's bad enough that we've had to bring the old Ladybug models out of mothballs to bolster our forces here."`
			`	"Even more unfortunately," he continues, "it seems that the overabundance of these pirates is the result of one particular individual's success. His ship, called the Black Lich, is in Waypoint with a stolen Geocoris that has been converted into some kind of semi-stable weapons platform and is now being used as a temporary base."`
			`	"This does give us an opportunity, however. If the kidnapped Hai are still on board, we might be able to rescue them, and if we can defeat the Black Lich, we may stem the tide of opportunists that are swarming through the wormhole.`
			`	"What do you say?"`
			`	The poor Captain seems desperate.`
			choice

> 				`	"We'll capture the geocoris and bring them safely home."`

Reviewer: ```suggestion
				`	"We'll capture the Geocoris and bring them safely home."`
```

					accept
	npc
		government "Hai"
		personality escort heroic disables
		fleet "Anti-Pirate Hai (No Wormhole Travel)" 5
	npc
		government "Pirate"
		personality staying disables plunders harvests
		system "Waypoint"
		fleet "Marauder fleet IX" 4
		fleet "Hired Guns" 1
	npc kill
		government "Pirate"
		personality staying marked target
		system "Waypoint"
		ship "Marauder Leviathan (Weapons)" "Black Lich"
	to complete
		or
			has "00 Hai Leaks Response 4B: declined"

> 			has "Hai Leaks Response 4A: failed"

Reviewer: I'm pretty sure in its current state you could get softlocked out of the story here if you capture the Geocoris but lose it before landing, since the completion conditions are you either blew up the Geocoris before capturing it or you successfully landed with the Geocoris as your flagship.

Author: Erm, you sure-sure? I seem to recall a decent effort to prevent that during a round of testing last year.

Reviewer: "00 Hai Leaks Response 4B: declined" is only set if you successfully captured the Geocoris and landed with it as a flagship.
"Hai Leaks Response 4A: failed" is only set if you failed to capture the Geocoris.
Neither of these conditions capture the scenario where you successfully capture the Geocoris but fail to land with it as a flagship (e.g. it was destroyed after capture or, God forbid, you sold it).

Author: If I recall correctly, one of the missions is meant to check that you have the outfit that the Geocoris carries, and if you do proceed and cancel the other mission as a get out of that loop situation.

			branch "saved geocoris2"
				has "00 Hai Leaks Response 4B: declined"
			`	With a final nod of thanks, he departs to attend to his business: leaving you free to go about your own.`
				decline
			label "saved geocoris2"
			`	With a final nod of thanks, he departs to attend to his business: leaving you free to wander the spaceport for a couple of hours until they release your new ship.`
				decline			
	
event "waypoint pirates vanquished"
	#Update pirate and anti-pirate spawns in Waypoint and Heia Due
	system "Waypoint"
		government "Hai"
		add fleet "Anti-Pirate Hai (Wormhole Travel)" 1000
		remove fleet "Marauder fleet IX"
		remove fleet "Anti-Pirate Hai (No Wormhole Travel)"
		remove fleet "Large Northern Pirates"
		remove fleet "Large Core Pirates"
	system "Heia Due"
		remove fleet "Marauder fleet IX"
		remove fleet "Large Northern Pirates"
		remove fleet "Large Core Pirates"

# It is impossible to implement a "capture and return npc" mission. 
# Once the ship is captured, the NPC is gone and the mission is failed.
# The NPC has an outfit used to determine if you managed to get it back safely.
mission "Hai Leaks Response 4A"
	name "Recover the Geocoris"

> 	description "Recover the stolen Geocoris supporting the pirates in Waypoint, and return to Allhome. (You will need to swap the Geocoris to your flagship, and depart and land again in order to progress the missions properly.)"

Reviewer: This is definitely not mergeable in this state. We'll either need to rework these missions to get the intended effect (I'd need to sit and think about it for a while to see if this is possible using existing mechanics), wait on new mechanics, or ditch the concept of capturing the ship; perhaps we just use a board conversation instead.

Author: All it needs is for #4561 to get resurrected and finished - and be able to properly register outfits on captured ships immediately when you first land with them.

Reviewer: "It works," isn't sufficient enough to be mergeable.

Author: ~After 2.5 years, with the PR that was meant to fix the issue being shelved, maybe it ought to be.~

	landing
	source "Allhome"
	waypoint "Waypoint"
	destination "Allhome"
	to offer
		has "Hai Leaks Response 3: done"
	npc capture
		government "Pirate"
		personality staying marked mute
		system "Waypoint"

> 		ship "Geodesic Geocoris (Phylactery)" "Phylactery"

Reviewer: So if we kill the Black Lich but leave the Geocoris does the Lich just keep respawning? /s

		dialog `After successfully boarding the Geocoris and defeating its pirate crews, you are greeted by the relieved captive Hai. You promise to escort their ship safely to Allhome. (You will need to swap the Geocoris to your flagship, and depart and land again in order to progress the missions properly.)`
	on fail

> 		dialog `You have failed to recover the Geocoris. This is not critical, but does have long-term consequences. If you want the most complete ending, revert to the autosave or another earlier snapshot of the game.`

Reviewer: I'd rather not have on fail dialogs for actions that aren't strictly a campaign failure.

Reviewer: Fail dialogs as they have been used in the past exist to tell you, "You've straight up failed this campaign and will need to reload an old save if you want to even finish it." Think "With this character's death, the thread of prophecy is severed. Restore a saved game to restore the weave of fate, or persist in the doomed world you have created." This is not that. This is, "Hey, there's some extra content you're able to access if you reload your save and complete the secondary objective," which feels very hand-holdy to me. If multiple endings exist based off of the character's choices and successes in earlier missions, they should be allowed to make those choices or successes (or failures) without being nudged by the game to go back and try things differently.

Author: Ok fairly pointed out.

		"heat dissipation" 0.4
		"fuel capacity" 800
		"cargo space" 1003
		"outfit space" 555
		"weapon capacity" 250
		"engine capacity" 134
		"energy capacity" 5200
		"shield generation" 6.0
		"shield energy" 8.8
		"hull repair rate" 0.8
		"hull energy" 0.8
		"hull heat" 1.2
		weapon
			"blast radius" 400
			"shield damage" 4000
			"hull damage" 2000
			"hit force" 6000
	outfits
		"Chameleon Anti-Missile" 2
		"Quad Blaster Turret" 2
		"Pulse Turret" 2
		
		"Quantum Keystone"
		"Boulder Reactor" 2
		"Liquid Helium Cooler"
		"Security Station" 2
		"Pulse Rifle" 30

		"A525 Atomic Steering"
		`"Biroo" Atomic Thruster`
		"AR120 Reverse Thruster" 2
		Hyperdrive

	turret 0 -82.5 "Pulse Turret"
	turret -95 -17 "Chameleon Anti-Missile"
	turret 95 -17 "Chameleon Anti-Missile"
	turret -29 3 "Quad Blaster Turret"
	turret 29 3 "Quad Blaster Turret"
	turret 0 91.5 "Pulse Turret"
	engine -95 41.5 0.9
	engine 95 41.5 0.9
	engine 1 154.5 0.9
	leak "leak" 40 50
	leak "flame" 40 80
	leak "big leak" 70 30
	explode "tiny explosion" 50
	explode "small explosion" 50
	explode "medium explosion" 50
	explode "large explosion" 50
	explode "huge explosion" 20
	"final explode" "final explosion large"

> 	description "This modified Geocoris has identical cargo capacity, but is modeled off one that was the result of an alien technology being merged with a hybrid Human-Hai operating system. It has additional shield emitters in a geodesic pattern over its main cargo wings, and a crucial element of the ship's system core is not manufactured, but grown from initial samples taken from the first occurrence. This means that some systems are built-in and unable to be removed, and also that it is substantially more expensive than its regular version. It is, however, significantly more survivable."

Reviewer: Grown from initial samples of what taken from the first occurrence of what?

Author: Bah, well that wasn't a problem before, and *was* going to be explained in a follow-up plot.

Author: There's a custom description for the actual variant you capture now - the original description will remain for if there is ever a way to acquire more of them.

	
ship "Geodesic Geocoris" "Geodesic Geocoris (Phylactery)"
	outfits
		"Electron Turret" 3
		"Chameleon Anti-Missile" 2
		"AR120 Reverse Thruster" 2
		"Boulder Reactor"
		"Hai Ravine Batteries"
		"Outfits Expansion"
		"Large Heat Shunt"
		"Quantum Keystone"
		"Pulse Rifle" 14
		`"Benga" Atomic Thruster`
		`"Bufaer" Atomic Steering`
		"Jump Drive"
		Hyperdrive
		"Failed Control System"
	turret "Electron Turret"
	turret "Electron Turret"
	turret "Chameleon Anti-Missile"
	turret "Chameleon Anti-Missile"
	turret "Electron Turret"

> mission "Hai Leaks Response 5"

Reviewer: Another instance where I'd prefer an on offer conversation to an on complete from the previous mission.

Author: Having just looked at this, I get ya, but I also note that the complete criteria and how that works is in here, and the conversation has branches based on that criteria. It's much more readable/trackable here.

> mission "Hai Leaks Response 5"

Reviewer: Another instance where I'd prefer an on offer conversation to an on complete from the previous mission.

	landing
	name "Talk to Hai authorities on <planet>"
	description "Meet Hai authorities on <destination>."
	source "Allhome"
	clearance
	destination "Hai-home"
	to offer
		has "Hai Leaks Response 4: done"

mission "Hai Leaks Response 6"

> 	name "Investigate rumours of kidnapped Hai"

Reviewer: ```suggestion
	name "Investigate rumors of kidnapped Hai"
```

	description "It's a statistical oddity, but Sayari's friend thinks someone is kidnapping Hai. Check spaceports in nearby parts of Hai and human space for evidence of a Hai trafficking operation."
	autosave
	landing
	source "Hai-home"
	clearance
	to offer
		has "Hai Leaks Response 4: done"
	on offer
		event "secret mountaintop"
		event "role-play fleet deception"
		event "slightly lost" 10
		# Hai like you a lot now, and as a result, you have permanent access to Hai-home:
		"reputation: Hai" >?= 100

> 		log `The working solution to the secrecy crisis for now is to have the Hai post a number of ships on the other side of the wormhole to play-act as an isolated furry role-play community. This is only a short-term solution. Asked by Sayari to look into a rumor about more kidnapped Hai. She does not consider it to be a particularly likely scenario, but her contacts insist.`

Reviewer: Not gonna lie, I feel like the furry joke got the airplay it needed during the Emerald Sword story and I'd rather not recycle it, let alone make it a major plot point of an extended storyline as opposed to the throwaway joke that it is in the Emerald Sword story. There it was funny, here I'm just cringing.

Author: To be fair, we did it first.

Reviewer: You sure? Emerald Sword was opened almost two months prior to this.

Author: I recall similarly.

		conversation

> 			`When you return to Hai-home, you are met by a group of relieved diplomats. Of all of them, Alondo seems to be the least tired and takes the time to let you know what's been going on. "Captain <last>! You've missed quite the palaver. Still, I doubt you lost much. We've spent four hours trying to explain to Sayari what a 'furry' is."`

Reviewer 2: _Slides figurative chair into room._
**Please. Tell us more.**

			choice
				`	"What? That could've been entertaining."`
				`	"Hmm. No, I probably didn't miss much."`
				`	"Oh, I could have told her all about it..."`
			`	He chuckles. "Anyway, I have a hunch that we'll need you before too long, but for now they're putting some Hai patrols out on the other side of the wormhole with a couple of Navy observers on board to 'play the part', as it were, of an exclusive role-play community, just chilling out here and having fun."`
			`	"I hear you've been instrumental in driving off some opportunistic pirates and helping stem the information leak. Come on, I'll buy you a drink on the others' behalf."`
			`	Some hours later, you leave the fancy cocktail bar Alondo took you to and begin making your way back to your ship to sleep. Your cabin by far seems to be the best place to not be bothered.`
			`	Partway there, a hooded figure suddenly falls in step beside you.`
			choice
				`	(Keep walking.)`
				`	(Ask them who they are.)`

> 			`	"Sssshhhh!" says the voice at your elbow, "do not react, and do not be alarmed, it is merely I."`

Reviewer: ```suggestion
			`	"Sssshhhh!" says the voice at your elbow. "Do not react, and do not be alarmed, it is merely I."`
```

> 			`	You look at the figure out of the corner of your eye and connect the whispered voice to the squirrelly muzzle poking out. To your surprise, it is Sayari.`

Reviewer: Why so secretive? Why can't this be told to us by Sayari in private under normal circumstances?

Author: The implication is that she doesn't have sufficient grounds to say it publicly as a representative of her government.

			choice
				`	"But what should I do about it?"`
			`	"You are well-known, and must be canny to have got where you are. A Hai in trouble might recognize you, or you might perceive them when others would not. It would be in the interests of both our people to be sure that there is no such issue. Would you look for me? For both our sakes?"`
			branch "about that"
				"hai slave prereq" >= 1
			choice
				`	"Sure, I can have a look around."`
					accept	
			label "about that"
			`	"Well, about that," you say, "I think I might have already come across some evidence of this before."`
			`	Sayari's hood nearly slips off as she looks up at you with a horrified expression. "Surely not?!"`
			`	You rush to reassure her. "Nothing concrete, of course. I can go and look to be sure."`
				accept
	to complete
		"hai slave prereq" >= 3
	on complete
		log `You found proof of more kidnapped Hai. Sayari's fears appear to have been entirely justified...`

mission "Hai Leaks Response 7"
	landing
	invisible
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Hai"
		not attributes "uninhabited"
	to offer
		has "Hai Leaks Response 6: active"

> 		"hai slave prereq" >= 3

Reviewer: Why are the missions that increment this in the next file? I think that either those missions should be moved into this file, or these missions that rely on this condition should be moved into that file, just so that we don't need to jump back and forth between files when trying to read through this.

Author: Because they're under Nomadic's copyright, the main reasons the files break up where they do is to do with who did what.

Reviewer: I'd rather not split content based on trying to get it under the copyright header that it matches. That's not something we've ever concerned ourselves with in the past. Content should go in the file that makes sense for where it is in the story, and the PR, through either the commit history or through mention in the description, should mention who did what.

Author: Mmm, I can see that. There is a legitimate conceptual divide between them though. You separate out from a structured A-B-C narrative to a "Go collect these in any order" narrative, and then back into an A-B-C narrative in the next file.

The file in question is also something I'd like to expand on in the future and it's much neater in its own file for that than tacked onto the end of the previous one.

# Copyright (c) 2020 MasterOfGrey
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A

> # PARTICULAR PURPOSE.  See the GNU General Public License for more details.

Reviewer 5: ```suggestion
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.
```
The standard copyright header was changed recently.

		conversation
			`Approaching Foundry, you are hailed on a private encrypted channel and directed to a private hangar where you are greeted personally by Syndicate CEO Alastair Korban.`
			`	He shakes your hand and smiles warmly. "Apologies, Captain <last>, do excuse the cloak and dagger, but in the current media climate you and I are both too well-known to be seen meeting without causing the rumor mill to grind out a new story."`
			`	"Now, allow me to introduce you to my protege, Terry Adrianopolous. She will be accompanying you as the Syndicate representative."`
			`	"Pleasure to meet you," she says with a broad smile. The young woman goes from calculating to gregarious in the blink of an eye as your attention shifts to her. She is pleasant and approachable, and very articulate, but from your brief glimpse beforehand you have no doubt she's cut from the sort of cloth that makes her future CEO material. You are surprised that a woman would have made it this far in the Syndicate, but if she has gotten this far, she must be very skilled at what she does.`

> 			`	"By the way," says Alastair, pausing briefly as he turns to leave, "The Navy and my security forces are working jointly to get some of our message relays up in the Far North to link through Ultima Thule. The pirates are wont to destroy them whenever they find them, but we should be able to rely on at least basic communication to get through if you need it, rather than flying back and forth for literally every question. That's all."`

Author: *except* here, beyond the scope of controlled/civilised space where it advantages them to be able to fly blind.

			label main
			`	You arrive at a very comfortable office and are received by a mature-looking Hai with the appropriate regalia of a diplomat. She holds out a paw to greet you and says, "My name is Sayari. I was the envoy to the human government a century ago."`
			`	The present media storm is all the evidence you need that the Republic has never mentioned the Hai, nor publicly acknowledged receiving an envoy from them. If they're in on the secret, then they're probably still trying to work out a suppression strategy and have been too busy to send an envoy quicker than your own trip took.`
			`	Sayari continues, "Our reports tell us that you are a well-known and respected individual in human space. We also know 'Hai' is quickly becoming a buzzword in your space. A few traders brought the news ahead of you."`
			`	She pauses and looks at you seriously. "We think it would be prudent for you to represent your people as our point of contact. Would you accept such a role?"`
			`	You give it a moment of thought.`
			choice
				`	"That seems practical."`
				`	"I don't really have an option to refuse, do I?"`

> 			`	"Fantastic, you understand the situation," she says, though her tone is much more business like than the wording would imply. "In that case, if you could please take this to Earth and communicate with your various governments we can begin a dialogue on how best to deal with this current scenario."`

Author: ```suggestion
			`	"Fantastic, you understand the situation," she says, though her tone is much more businesslike than the wording would imply. "In that case, if you could please take this to Earth and communicate with your various governments we can begin a dialogue on how best to deal with this current scenario."`
```

			`	You suspect it might be time to go visit your Hai friends again and see what's going on.`
		fail
	
mission "event: hai secret leaks B"
	landing
	source
		government "Free Worlds"
		attributes "spaceport"
	to offer
		has "event: hai secret leaks"
		not "event: hai secret leaks A: offered"
	on offer
		event "pirates investigate leaks"
		conversation
			`You are relaxing at the spaceport while your ship refuels after conducting a spot of business. On the viewscreen is some talk show or other, not something you specifically paid attention to but apparently a quite popular one out of Tarazed.`
			`	A young and fresh-faced man who looks incredibly ill at ease has just been brought out as a surprise guest by the host Jonathon Camembert.`
			`	"So now in our surprise segment let me introduce you to Ronald. He once crewed on one of these big trade ships but after seeing what he saw, he just had to come foward!"`

> 			`	The host stresses the word 'had' so much his voice squeeks but the audience's attention has certainly been gained.`

Author: So in this particular case at the start the style is a deliberate choice to emulate the way it reads when the news breaks for the terrorist bombing and the Free Worlds declaring their independence. I'll keep an eye out for it later on though, cause it's probably not the best later in the story.

			`	Even the spaceport is symmetrical: a ring of hangars, facing outwards, at evenly spaced intervals, with warehouses and living quarters inside the ring.`
			`	While the layout is utilitarian, it is an incredibly comfortable place to live, and the Hai have given their individual community spaces quite a lot of character and life. It is, overall, an excellent example of an idealised Hai world, built in the presence of abundant resources.`
			`	As you are wandering around and getting a feel for the place you overhear someone talking about an apparently complete ringworld in Alta Hai. It definitely sounds like something you should check out.`
		
	#Stop 2 Alta Hai (Alta Hai Ringworld)
mission "leaks tour 2"
	name "Check out the ringworld Alta Hai"
	description "Go to Stonebreak and learn more about the Hai's worlds."
	landing
	source
		planet "Stonebreak"
	destination "Alta Hai"
	to offer
		has "leaks tour 1: done"
	on complete
		conversation

> 			`This is a complete Quarg ringworld, the product of millennia of construction effort which has consumed every other stellar object in this system in order to provide the raw materials. The inner edge of the ring is filled with solar panels, hydroponic gardens, living quarters, and sunlit common spaces. The outer edge contains hangars and docks for ships, and storage facilities for cargo.`

Author: This part of the mission chain is explicitly to introduce you to some of the most significant place descriptions for players that are having first contact at this point. The duplication is intentional.

			`	As you're looking around you find a Quarg to query, "Isn't this area Hai space."`
			`	The Quarg explains, "This is the boundary where the Quarg interact with the Hai. From here we help their civilisation if it needs it."`
			`	This explanation seems cryptic but it's the best one you're able to get.`
			`	Before you go you ask, "If you help the Hai why don't you help with the rebels I've been hearing about?"`
			`	The Quarg responds, "The Hai and their rebels are the same. You should travel to Darkcloak in Wah Yoot to understand."`
			`	It seems like there's nothing else for it but to see for yourself.`

	#Stop 3 Wah Yoot (Darkcloak)
mission "leaks tour 3"
	name "Meet the Rebels on Darkcloak"
	description "Go to Darkcloak and learn more about the Hai's conflict."
	landing
	source
		planet "Alta Hai" 
	destination "Darkcloak"
	to offer
		has "leaks tour 2: done"
	on complete
		conversation

> 			`You had to bribe your way through just to land without being blown up. These Unfettered Hai do not seem friendly, but it's time to find out what they're about.`

Author: This mission will only come up if you haven't had first contact with them yet - so you will in fact have had to bribe your way through if you're doing this particular mission.

		government "Hai"
		add fleet "Small Hai" 1200
		add fleet "Large Hai" 2500
		remove fleet "Marauder fleet IX" 900
		remove fleet "Large Northern Pirates" 600
		remove fleet "Large Core Pirates" 600
	system "Heia Due"
		remove fleet "Marauder fleet IX" 900
		remove fleet "Large Northern Pirates" 600
		remove fleet "Large Core Pirates" 600

		
mission "leaks response 4a"
	landing
	invisible
	source "Allhome"
	destination "Allhome"
	to offer
		has "leaks response 3: done"
	npc board save
		government "Pirate"
		personality staying nemesis
		system "Waypoint"
		ship "Geodesic Geocoris (Phylactery)" "Phylactery"
		dialog
			`After successfully boarding the Geocoris and defeating its pirate crews you are greeted by the relieved captive Hai. You promise them that they will be safely on Allhome before long.`
	on fail
		dialog `You have failed to recover the Geocoris. This is not critical, but does have long-term consequences. If you want the heroic ending, revert to the autosave or another earlier snapshot of the game.`
	on complete
		event "returned geocoris" 365
		#The Geodesic Geocoris may become available to purchase specially later on.
		
event "returned geocoris"
		

> ship "Geodesic Geocoris"

Author: I have noticed that obscure ships unlocked by specific mission chains are often int he mission files instead. However, if you think so I'll move the main one to Hai Ships and the variant can just stay here. I think I still need to take another pass at the stats for it too.

> ship "Geodesic Geocoris"

Reviewer 6: That one is probably fine, but the long list I made for the temporary Human Alliance probably needs to move. I will move them in the PR I'll send you in a few days.

			`	Pondering your questions, you find a Quarg and ask, "So this isn't Hai space then, but it's a dead end?"`
			`	The Quarg explains, "Much like in Enif in human space, this is where we live as friends to the Hai."`
			`	This explanation seems typically Quarg, but it's the best one you're able to get. Before you go, you ask, "Do you know about the rebels I've been hearing about?"`
			`	The Quarg responds, "The Hai and their rebels are the same. You should travel to Darkcloak in Wah Yoot to understand."`
			`	It seems like there's nothing else for it but to see for yourself.`
				accept
	
	#Goto Stop 4 Wah Ki (Cloudfire)
	# Must happen before unfettered first contact
mission "00 Hai Tour [4] Cloudfire"
	landing
	name "Find out about the Rebels on Cloudfire"
	description "Go to Cloudfire and compare the Hai's side of the conflict."
	source "Darkcloak" 
	destination "Cloudfire"
	to offer
		or
			has "Hai Witness Unfettered [1]"

> 			has "ZZ Hai Tour [3] Meet Unfettered: done"

Reviewer 7: This didn't fire for me, I got the usual history lesson and request for jump drives instead. After flying around the galaxy for a while, I figured something was wrong and temporarily changed `has "Hai Witness Unfettered [1]"` to `has "Hai Witness Unfettered [1]: done"` so I could trigger this by landing on Cloudfire again.

Would it make sense to give "Hai Witness Unfettered [1]" a stopover on Darkcloak with Cloudfire as the destination and then reuse the completion conversation of this mission there as well? Hopefully that would avoid any weird interactions with the normal unfettered stuff and allow the next mission to pick up as it should.

Author: Congrats you appear to have found a bug that slipped through initial testing, this will be fixed.

		conversation
			`On approach to the spaceport on Earth, you request to speak to an official of significant standing on diplomatic business. Thanks to your renown from the Pug war, you are able to shortcut objections stemming from you yourself not being a diplomat. The diplomatic ID from the box Sayari gave you doesn't hurt either.`
			`	The Republic Deputy Minister for Foreign Affairs meets you in a private room at the spaceport.`
			`	"Captain <last>," he steps around the table to grasp your hand in a firm handshake. "Xilin Yang. It's an honor to meet a war hero like yourself."`
			choice
				`	"It was hard-fought, but it was necessary."`
				`	"Thanks, but it wasn't as dramatic as they made it out to be. I'm just happy to have helped where I could."`
					goto humble
			`	You play the part and try to act noble. Mr. Yang seems more than happy to embody respect for the hero of his preferred narrative.`
			`	"You're every inch what I expected," he assures you before moving to take his seat.`
				goto sit
			label humble

> 			`	You shrug, palms up, and explain, "You should listen to the xenoanthropologists. The Pug were just here to test us, to make us really consider what mattered and come together."`

Reviewer 7: This felt like a bit of a non sequitur, and I was confused for a while until I realised he was talking about the war with the Pug rather than the war with the Free Worlds. Perhaps it would make more sense if he described you as "the hero who defeated the Pug" rather than "a war hero"?

Author: Given that most of the fighting was against the Pug and that there were only a small handful of conflicts between the FW and Navy, that kinda seemed obvious to me. I'll see about making this more clear though.

Reviewer 7: For me, the length of time spent in the "FW vs Republic" part of the war outweighs the relatively brief "everyone vs Pug" segment, regardless of the amount of actual combat involved. Either way, the player could have put the game down for a month between sessions and forgotten a lot of stuff -- I do this myself rather more often than I like.

			`	For a moment his expression falters, and then he seems almost relieved. "You may be a better man than I expected to meet."`
			`	He offers you a genuine smile as he takes his seat.`
			label sit
			`	You sit down opposite him and explain how the Hai have proposed that you be their official contact point for humanity on account of your fame from the war.`
			`	He asks a few questions for clarity, and when you're done he requests you wait while he makes a call. He leaves the room for a long while, and at one point an aide comes in and offers you refreshments while deliberations continue elsewhere. Much to your surprise the fruity beverage is actually quite light and not too sweet, reminding you briefly that most of humanity's food styles originated here.`
			`	Eventually Mr. Yang returns. "So sorry for that. I hope you were looked after suitably while you waited."`

> 			`	Once you nod your assent, he continues, "Well, obviously the Republic knows about the Hai, but for the moment our official "line" is that the government does not trade in rumors. However, an appropriate response is clearly needed. Republic Intelligence briefed the Cabinet of the Free Worlds Senate just yesterday. They have demanded that they have a diplomatic representative along for any decisions. This is already standard practice for the Syndicate. If you are willing, I will come with you to meet two individuals known as Katya and Alastair on Deep and Foundry respectively. I believe you already know them?"`

Reviewer 7: I would expect a diplomat discussing official business to use full names, and it took be a while to figure out who Alastair was. Maybe introduce them with full names and titles to remind the player who they are?

Also, the only meeting I actually remember with Alastair Korban (from the Reconciliation branch, I've never played Checkmate) involved him resigning as CEO, so maybe it would be appropriate to allow the player to ask why he's still in power?

Author: The player gets additional explanation when they get to Alastair if they played Recon, it wouldn't necessarily make sense for Mr. Yang to know the details of internal Syndicate politics.

Reviewer 7: Yeah, an explanation isn't really *that* necessary here. A reminder of who the characters are (especially Alastair, who doesn't get much screen time) would solve the real issue.

		conversation
			`Approaching Foundry, you are hailed on a private encrypted channel and directed to a private hangar where you are greeted personally by Syndicate CEO Alastair Korban.`
			`	He shakes your hand and smiles warmly. "Apologies, Captain <last>, do excuse the cloak and dagger, but in the current media climate you and I are both too well-known to be seen meeting without causing the rumor mill to grind out a new story."`
			`	"Now, allow me to introduce you to my protege, Terry Adrianopolous. She will be accompanying you as the Syndicate representative."`
			`	"Pleasure to meet you," she says with a broad smile. The young woman goes from calculating to gregarious in the blink of an eye as your attention shifts to her. She is pleasant and approachable, and very articulate, but from your brief glimpse beforehand you have no doubt she's cut from the sort of cloth that makes her future CEO material. You are surprised that a woman would have made it this far in the Syndicate, but if she has gotten this far, she must be very skilled at what she does.`

> 			`	"By the way," says Alastair, pausing briefly as he turns to leave, "The Navy and my security forces are working jointly to get some of our message relays up in the Far North to link through Ultima Thule. The pirates are wont to destroy them whenever they find them, but we should be able to rely on at least basic communication to get through if you need it, rather than flying back and forth for literally every question. That's all."`

Author: Nope, you may want to look up the definition of "wont".

> 			`	"By the way," says Alastair, pausing briefly as he turns to leave, "The Navy and my security forces are working jointly to get some of our message relays up in the Far North to link through Ultima Thule. The pirates are wont to destroy them whenever they find them, but we should be able to rely on at least basic communication to get through if you need it, rather than flying back and forth for literally every question. That's all."`

Reviewer 8: Wont is a pretty uncommon word and I'm sure there are going to be more people confused by it. Maybe change it to something else?

> 			`	"By the way," says Alastair, pausing briefly as he turns to leave, "The Navy and my security forces are working jointly to get some of our message relays up in the Far North to link through Ultima Thule. The pirates are wont to destroy them whenever they find them, but we should be able to rely on at least basic communication to get through if you need it, rather than flying back and forth for literally every question. That's all."`

Author: Characters that would be highly educated and in the habit of talking concisely will sometimes use uncommon words. I don't believe in arbitrarily simplifying language, particularly where it pertains to characterisation, in a text-based game. This is the sort of medium that should be able to occasionally introduce people to words they might not encounter very often.

			branch "saved geocoris2"
				has "00 Hai Leaks Response 4B: declined"
			`	With a final nod of thanks, he departs to attend to his business: leaving you free to go about your own.`
				decline
			label "saved geocoris2"
			`	With a final nod of thanks, he departs to attend to his business: leaving you free to wander the spaceport for a couple of hours until they release your new ship.`
				decline			
	
event "waypoint pirates vanquished"
	#Update pirate and anti-pirate spawns in Waypoint and Heia Due
	system "Waypoint"
		government "Hai"
		add fleet "Anti-Pirate Hai (Wormhole Travel)" 1000
		remove fleet "Marauder fleet IX"
		remove fleet "Anti-Pirate Hai (No Wormhole Travel)"
		remove fleet "Large Northern Pirates"
		remove fleet "Large Core Pirates"
	system "Heia Due"
		remove fleet "Marauder fleet IX"
		remove fleet "Large Northern Pirates"
		remove fleet "Large Core Pirates"

# It is impossible to implement a "capture and return npc" mission. 
# Once the ship is captured, the NPC is gone and the mission is failed.
# The NPC has an outfit used to determine if you managed to get it back safely.
mission "Hai Leaks Response 4A"
	name "Recover the Geocoris"

> 	description "Recover the stolen Geocoris supporting the pirates in Waypoint, and return to Allhome. (You will need to swap the Geocoris to your flagship, and depart and land again in order to progress the missions properly.)"

Author: The behaviour has been weird and inconsistent in the past. Someone was planning to replace it with a more detailed range of commands, but for now this is just the most reliable way to make sure the missions progress successfully.

			`	Others...`
			`	You sigh deeply as you watch some paranoid fringe world politician demand that the edges of human space be better understood, that threats and even potential threats must clearly exist beyond what is already known and that it was the responsibility of the government to find out and inform the people.`
			`	Your thoughts, you keep to yourself.`
				decline

event "hai secret leaks"

conversation "hai reveal free worlds news"
	`You are relaxing at the spaceport while your ship refuels after conducting a spot of business. On the viewscreen is some talk show or other, not something you specifically paid attention to but apparently a quite popular one out of Tarazed.`
	`	A young and fresh-faced man who looks incredibly ill at ease has just been brought out as a surprise guest by the host Jonathon Camembert.`

> 	`	"So now in our surprise segment let me introduce you to Ronald. He once crewed on one of these big trade ships but after seeing what he saw, he just had to come foward!"`

Author: The use of "these" is colloqial to imply ownership. Tarazed is a major port that has a major shipyard so it's used in the sense of "*these* ships that are part of our lives", as opposed to a backwater which would say "those ships that visit occasionally".
Otherwise, good catch. 👍

	`	The host stresses the word 'had' so much his voice squeeks but the audience's attention has certainly been gained.`
	`	"But first, lets take a look at a clip."`
	`	You raise your eyebrows with surprise.`
	branch "hai knowledge"
		has "First Contact: Hai: offered"

> 	`	A clip of alien space ships in combat with weapons quite unlike anything produced in human space is shown, you've seen things, but not this.`

Author: The 'quite' is just for emphasis, all Hai stuff is unlike our own, their Ion cannons are *quite* unlike our own in that they are not only different in design, but different on an even more fundamental scientific level than mere design. It's to hint at whole new types of weapons that operate differently than those in human space.

> 	`	When it's finished the young man, who is very well presented but clearly has a bit of the paranoid about him that is plain to see even through a screen, is encouraged to talk about where he got the footage. He talks at length about an ongoing conflict that threatens to spill over and drag all of humanity back to war. A war waged by an alien species known as the 'Hai'.`

Author: I'm going to disagree here, it's a standard use of a clause and the intent is to paint a picture in the reader's mind about both the style of encouragement that is used, and the manner in which he goes about explaining what he's seen.
It's not as simple as them asking him what he saw and him laying it out clean and logically, it's a back and forth, without a strict structure to the presentation, with the man jumping back to details missed earlier as they come up. It characterises the interview without having to go into all the detail of exactly how it occurs.

	`	That such news should come out so soon after the Pug smacks of a hoax. A quick glance at the clock shows you only have ten minutes to wait so you stick around for the full segment out of curiosity.`
	`	Right on schedule the news returns with the severe anchorwoman. It turns out that the crewman in question had been picked up by a minor media outlet on Tarazed. He had apparently travelled many weeks before deciding to seek the media in order to avoid being shut down by Republic Intelligence if he contacted media elsewhere. The man is fairly young, and very well presented, though he has a bit of the paranoid about him that is plain to see even through a screen.`
	branch "hai knowledge"
		has "First Contact: Hai: offered"
	`	A clip of alien space ships in combat with weapons quite unlike anything produced in human space is shown, you've seen things, but not this.`
	`	It is only after this that the anchorwoman reveals that, while this was initially treated as a hoax, one of their own reporters posed a timely question to a well-respected trader and caused quite the stir when he slipped up and confirmed the existance of the aliens known as 'Hai'.`

> 	`	This is, of course, ridiculous... but maybe it would be good to go investigate this for yourself. You listen in to collect the relevant details for future reference.`

Author: I feel like media in the Republic isn't likely to be all that respectable, and in all honesty I would expect its reputation would've declined during the war. That said, it's mostly meant to be more like a dismissal of the idea that an entire alien race that you can just fly to in a few days could have remained hidden *while at war* for any period of time. It's a juxtaposition of reality.

		accept	
	label "hai knowledge"
	`	A clip of alien space ships in combat with weapons quite unlike anything produced in human space is shown, and you recognise shield beetles hammering away at each other.`
	`	It is only after this that the anchorwoman reveals that, while this was initially treated as a hoax, one of their own reporters posed a timely question to a well-respected trader and caused quite the stir when he slipped up and confirmed the existance of the aliens known as 'Hai'.`
	`	You suspect it might be time to go visit your Hai friends again and see what's going on.`
		accept

conversation "What is going on in Hai space? (Have not met Unfettered.)"

> 	`Landing in Hai space draws a much different type of attention than expected. Only recently you've begun to acclimate yourself to getting positive attention. As a war hero you're something of a minor celebrity now back in human space, but here suddenly mothers are keeping tight grips on their children and passers-by look upon you with a hint of wariness.`

Author: This is incorrect.
http://www.aimpublishing.com/index.php?main_page=document_general_info&products_id=1303

	`	You realise that the few humans who have settled here among the Hai are all wearing pink at the moment, and when you catch the eye of one lady who looks friendly you greet them to ask a few questions.`
	choice
		`	"Why does everyone seem so afraid?"`

> 	`	She shakes her head and wrinkles her nose with a look of distaste, "Within about three days of hearing that the Hai were being splashed all over major news networks back home a whole heap of curious pirates have chanced the wormhole en mass. The local security forces on patrol at the time got blown out of the sky by a surprise attack and everyone has been a bit shaken up since."`

Author: I have noticed this comma/period thing in some other places in ES, but it flies in the face of what I've been taught and all usage I've observed elsewhere. If you have a source on this I'd appreciate it because otherwise I'm inclined to disagree.

> 	`	"Oh that's awful!" you say.`

Author: Why does it need a pause?

	on offer
		set "asked to investigate hai leak"
		event "news: hai video clip published"
		event "pirates investigate leaks"
		conversation "hai reveal free worlds news"
	
	
event "pirates investigate leaks"
	system "Waypoint"
		government "Pirate"
		add fleet "Marauder fleet IX" 2000
		add fleet "Anti-Pirate Hai (No Wormhole Travel)" 700

#This branch is if you have met the Hai but not the Unfettered
	#Goto Stop 3 Wah Yoot (Darkcloak) and pick up at Tour 4 mission.
mission "Hai Witness Unfettered [1]"
	name "Witness the Unfettered"
	description "Turns out this endless conflict has some truth to it, go see where it's happening."
	landing
	source "Allhome"
	destination "Darkcloak"
	to offer
		or
			has "ZZ Event Hai Secret Leaks: RSH: done"
			has "ZZ Event Hai Secret Leaks: FWH: done"

# This fires if you haven't met the Hai, to trigger the entire tour.
# It must happen after the first contact mission.
	#Goto Stop 1 Imo Dep (Stonebreak)
	#This follows immediately after the First Contact text - no other convo is required.
mission "ZZ Hai Tour [1] Stonebreak"
	landing
	name "You're summoned to <planet>"
	description "Go to <destination> and learn more about the Hai's worlds."

> 	source "Allhome"

Author: This particular mission ties off the ones which have Allhome as the destination, so people would have to actively fly past their mission goal and go somewhere else to trigger them out of order.

> 	source "Allhome"

Reviewer 3: Oh! Didn't even notice that. In that case, I agree. My bad!

			`	Even the spaceport is symmetrical: a ring of hangars, facing outwards, at evenly spaced intervals, with warehouses and living quarters inside the ring.`
			`	While the layout is utilitarian, it is an incredibly comfortable place to live, and the Hai have given their individual community spaces quite a lot of character and life. It is, overall, an excellent example of an idealised Hai world, built in the presence of abundant resources.`
			`	As you are wandering around and getting a feel for the place you overhear someone talking about an apparently complete ringworld in Alta Hai. It definitely sounds like something you should check out.`
				accept

	#Goto Stop 3 Wah Yoot (Darkcloak) - must happen after Quarg first contact
mission "ZZ Hai Tour [3] Meet Unfettered"
	landing
	name "Meet the Rebels on <planet>"
	description "Go to <destination> and learn more about the Hai's conflict."
	source "Alta Hai"
	destination "Darkcloak"
	to offer
		has "Hai Tour [2] Alta Hai: done"
	on offer
		conversation

> 			`This is a complete Quarg ringworld, the product of millennia of construction effort which has consumed every other stellar object in this system in order to provide the raw materials. The inner edge of the ring is filled with solar panels, hydroponic gardens, living quarters, and sunlit common spaces. The outer edge contains hangars and docks for ships, and storage facilities for cargo.`

Reviewer 3: I asked about that last sentence in the discord [here](https://discordapp.com/channels/251118043411775489/266345072554016768/746142672959569951). Below is Marksman's recommendation for the line.
```suggestion
			`This is a complete Quarg ringworld, the product of millennia of construction effort which has consumed every other stellar object in this system in order to provide the raw materials. The inner edge of the ring is filled with solar panels, hydroponic gardens, living quarters, and sunlit common spaces. The outer edge contains hangars and docks for ships while housing storage facilities for cargo.`
```

> 			`	Pondering your questions, you find a Quarg to query, "So this isn't Hai space then, but it's a dead end?"`

Reviewer 3: ```suggestion
			`	Pondering your questions, you find a Quarg and ask, "So this isn't Hai space then, but it's a dead end?"`
```
```suggestion
			`	Pondering your questions, you find a Quarg to query. "So this isn't Hai space then, but it's a dead end?"`
```
Either is grammatically correct; your choice.

			`	The Quarg explains, "Much like in Enif in human space, this is where we live as friends to the Hai."`
			`	This explanation seems typically Quarg, but it's the best one you're able to get.`

> 			`	Before you go you ask, "Do you know about the rebels I've been hearing about?"`

Reviewer 3: ```suggestion
			`	Before you go, you ask, "Do you know about the rebels I've been hearing about?"`
```

			`	The Quarg responds, "The Hai and their rebels are the same. You should travel to Darkcloak in Wah Yoot to understand."`

> 			`	It seems like there's nothing else for it but to see for yourself.`

Reviewer 3: This is another spot that desperately needs to be combined into fewer paragraphs.

				accept
	
	#Goto Stop 4 Wah Ki (Cloudfire)
	# Must happen before unfettered first contact
mission "00 Hai Tour [4] Cloudfire"
	landing
	name "Find out about the Rebels on Cloudfire"
	description "Go to Cloudfire and compare the Hai's side of the conflict."
	source "Darkcloak" 
	destination "Cloudfire"
	to offer
		or
			has "Hai Witness Unfettered [1]"
			has "ZZ Hai Tour [3] Meet Unfettered: done"
	on offer
		conversation

> 			`You had to bribe your way through just to land without being blown up. These Unfettered Hai do not seem friendly, but it's time to find out what they're about.`

Reviewer 3: You might consider adding a sentence somewhere clarifying that the player should go to Cloudfire next.

Author: This mission only occurs if you haven't got Unfettered first contact. In which case the first contact mission sends you to cloudfire quite smoothly in this scenario.

Reviewer 3: This is complex and it's been a while since I had a grasp on how everything flowed, but as long as the player is explicitly or very implicitly told to go to specifically Cloudfire, then it should be good.

				accept
	on complete
		conversation
			`You dodge the conflict in space and come in to land. Cloudfire's atmosphere is unusually hazy, with almost constant cloud cover, making agriculture unproductive here even though it is a warm and damp environment. There are several major cities here that were abandoned centuries ago, due to conflict with the Unfettered. The one major city that remains is the closest thing the Hai have to a military base, with new supplies of weapons and ammunition constantly pouring in from the rest of Hai space.`
			`	The spaceport has massive hangars and advanced repair facilities designed to accommodate the largest Hai warships. Where a fleet has just returned from a recent battle, the hangars are lit with flashes from robotic welders as damaged hull plates are swapped out for fresh ones and weakened joints are reinforced. The actual process of building these warships, however, takes place on the Hai homeworld, where the facilities will not be vulnerable to attack.`

> 			`	It does not take long to realise that this conflict really is constant, even if the friendlier Hai on this side seem to be maintaining their defence without apparently taxing their civilisation to any great extent.`

Reviewer 3: ```suggestion
			`	It does not take long to realize that this conflict really is constant, even if the friendlier Hai on this side seem to be maintaining their defense without apparently taxing their civilization to any great extent.`
```
_screams in bald eagle_

# Both tours merge here
mission "Hai Tour [5] Tour Ends"
	landing
	name "Head directly to Hai-home"
	description "Go to Hai-home and meet whoever wants to speak with you."
	source "Cloudfire"
	destination "Hai-home"
	clearance
	to offer
		has "00 Hai Tour [4] Cloudfire: done"
	on offer
		conversation
			branch "skipped tours"
				has "Hai Witness Unfettered [1]"

> 			`After looking around Cloudfire for some time you stop in the spaceport for a moment to absorb everything you've seen. A screen is playing through news on the wall nearby and your ears suddenly perk up at the word "marauder".`

Reviewer 3: ```suggestion
			`After looking around Cloudfire for some time, you stop in the spaceport for a moment to absorb everything you've seen. A screen is playing through news on the wall nearby, and your ears suddenly perk up at the word "marauder".`
```

> 			`	"Yesterday a marauder terrorised the inhabitants of Allhome. They arrived by posing as a trader to investigate the rumours. Hai security failed to identify the danger and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew kidnapped nearly four dozen Hai and hijacked a Geocoris before making off with their spoils."`

Reviewer 3: ```suggestion
			`	"Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew kidnapped nearly four dozen Hai and hijacked a Geocoris before making off with their spoils."`
```
Added that bit in because any news source wouldn't assume its viewers know what rumors are being referred to.

				goto "that's awful"
			label "skipped tours"

> 			`Soon after you land on <planet>, you notice a news broadcast on a nearby wall, "Yesterday a marauder terrorised the inhabitants of Allhome. They arrived by posing as a trader to investigate the rumours. Hai security failed to identify the danger and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew kidnapped nearly four dozen Hai and hijacked a Geocoris before making off with their spoils."`

Reviewer 3: ```suggestion
			`Soon after you land on <planet>, you notice a news broadcast on a nearby wall: "Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew kidnapped nearly four dozen Hai and hijacked a Geocoris before making off with their spoils."`
```
You could also use a period instead of a semicolon after `wall`, but the comma needs to be changed to one of those two.

				goto "that's awful"
			label "that's awful"

> 			`	At a table nearby you hear a Hai staff member exclaim, "Oh that's awful!"`

Reviewer 3: ```suggestion
			`	At a table nearby you hear a Hai staff member exclaim, "Oh, that's awful!"`
```

> 			`	The rumours must be spreading further by the day you think, no doubt something will soon have to be done.`

Reviewer 3: ```suggestion
			`	The rumors must be spreading further by the day. There's no doubt even more problems than this marauder will follow.`
```

> 			`	This thought has no sooner crossed your mind when your solitude is interrupted by a small group of somewhat official-looking Hai.`

Reviewer 3: ```suggestion
			`	This thought has no sooner crossed your mind when your solitude is interrupted by a small group of official-looking Hai.`
```
`somewhat` felt unnecessary.

> 			`	"Are you Captain <last>?" one of them asks you, "Don't worry, you're not in trouble or anything."`

Reviewer 3: ```suggestion
			`	"Are you Captain <last>?" one of them asks you. "Don't worry, you're not in trouble or anything."`
```

			`	You nod your affirmation.`

> 			`	"Your reputation precedes you Captain. As a person of note at home who we hear has now been travelling around Hai space, you are presently being treated as an Envoy."`

Reviewer 3: ```suggestion
			`	"Your reputation precedes you, Captain. As a person of note among humanity who we hear has now been travelling around Hai space, you are presently being treated as an Envoy."`
```

> 			`	For a second you wonder if it's a joke but they look as serious as Hai can look.`

Reviewer 3: ```suggestion
			`	For a second you wonder if it's a joke, but they look as serious as Hai can look.`
```

			choice

> 				`	"Oh I really don't think that's necessary."`

Reviewer 3: ```suggestion
				`	"I really don't think that's necessary."`
```
Having both options start with `Oh` feels too much like you're impressing specific speech patterns onto the player. Best to avoid that from my perspective.

					goto nevertheless

> 				`	"Oh, I'm honoured, thank you."`

Reviewer 3: ```suggestion
				`	"Oh, I'm honored. Thank you."`
```

> 			`	"If you have any stake in the current information crisis please head directly to Hai-home. Someone would very much like to talk to you."`

Reviewer 3: ```suggestion
			`	"If you have any stake in the current information crisis, please head directly to Hai-home. Someone would very much like to talk to you."`
```

				goto main
			label nevertheless
			`	"You may not think it's necessary, but sometimes that is the price of fame."`

> 			`	The officious-looking Hai shrugs, "In any event please head directly to Hai-home. Someone would very much like to talk to you."`

Reviewer 3: ```suggestion
			`	The official-looking Hai shrugs. "In any event, please head directly to Hai-home. Someone would very much like to talk to you."`
```

			label main

> 			`	They leave without waiting for your agreement, but then, you did come here to see what was going on.`

Reviewer 3: This conversation as well desperately needs some paragraphs to be combined where possible.

				accept

#All prior branches merge here.
	
mission "Hai Leaks Response 1"
	landing
	name "Hai message to <planet>"
	description "Take the diplomatic ID from the box from Sayari to <destination>."
	source "Hai-home"
	destination "Earth"
	to offer
		or
			has "Hai Tour [5] Tour Ends: done"
			has "ZZ Event Hai Secret Leaks: RSHU: done"
			has "ZZ Event Hai Secret Leaks: FWHU: done"
	on offer

> 		log "People" "Sayari" `Ambassador Sayari, a Hai, served as their envoy to the human government a century ago (although no public records of this exist in human space). Her experience has placed her in charge of coordinating the Hai response to the crisis of their continued secrecy.`

Author: This is the one spot as I recall that introduces a real quirk with existing saves. Sayari's log entries will be out of order and one of them won't strictly reflect what the current state is. However, for new saves they'll occur in order because the planet you land on to start wanderers won't let you land now until you complete an interaction in HR6.

> 		log "People" "Sayari" `Ambassador Sayari, a Hai, served as their envoy to the human government a century ago (although no public records of this exist in human space). Her experience has placed her in charge of coordinating the Hai response to the crisis of their continued secrecy.`

Reviewer 3: Interesting. I don't know if there's really a good resolution for this issue, but I imagine most of our core playerbase using the same file over multiple releases wouldn't mind _that_ much, especially since there's no simple fix.

> 		log `The Hai are very real, and the worlds of humanity seem poised to become aware of this but the Hai would like to avoid this. As a person of note with respect in all corners of human space the Hai have asked you to assist in representing their interests where travel to human space is required as a somewhat neutral party.`

Author: So they have contact, they just don't have formal representation in a continuous sense. 
Also, to avoid having separate logs if you learned about the Hai through these events, how does this sound:
`log `The Hai are very real, and human worlds seem poised to become aware of their existence. The Hai would like to avoid this and requested assistance in representing their interests where travel to human space is required. Their desire was for a human, yet somewhat neutral, party.``

> 		log `The Hai are very real, and the worlds of humanity seem poised to become aware of this but the Hai would like to avoid this. As a person of note with respect in all corners of human space the Hai have asked you to assist in representing their interests where travel to human space is required as a somewhat neutral party.`

Reviewer 3: That's fine, but I would like to suggest taking the last half of the first sentence and making it a little more in-line with the rest of the logs. Feel free to use it if you like, or just keep your own suggestion; that works too.
```suggestion
		log `The Hai are very real, and human worlds seem poised to become aware of their existence. The Hai would like to avoid this. Agreed to assist by representing their interests where travel to human space is required. Their desire was for a human, yet somewhat neutral, party.`
```

		conversation
			`As you come into land on Hai-home you find that a reasonable security detachment has been assigned to your designated landing berth. The Hai aren't usually ones to be concerned about such things, but you suspect that as an apparent "war hero" that this may be more for the benefit of appearances on your part than anything else.`
			`	Word of your status in human space seems to have spread ahead of you, especially with you being known on many Hai worlds as one of the few significant human visitors. A small crowd of Hai have gathered to peek at you on the short walk from the Spaceport to the government building you are led to.`
			branch karma
				karma > 1

> 			`	As you are making the walk you hear questions posed and rumours stated. Occasionally you hear words like "survivor" and "dangerous" thrown in amongst the babble.`

Author: I am inclined to instead add some scattered humans into the crowd that might be the cause for questions to be asked in the standard human tongue.

> 			`	As you are making the walk you hear questions posed and rumours stated. Occasionally you hear words like "survivor" and "dangerous" thrown in amongst the babble.`

Reviewer 3: We're on Hai-home, right? Isn't there little to no human presence here?

> 			`	As you are making the walk you hear questions posed and rumours stated. Occasionally you hear words like "survivor" and "dangerous" thrown in amongst the babble.`

Author: Correct, but given how well travelled Hai space is I assume *little* is the correct term rather than none, and what event is more likely to bring them out to see than the intersection of the politics of their old home and their new?

> 			`	As you are making the walk you hear questions posed and rumours stated. Occasionally you hear words like "survivor" and "dangerous" thrown in amongst the babble.`

Reviewer 3: Even if there are _some_ humans on Hai-home, I would imagine the Hai would so far outnumber them that the general babble from everyone would drown out any minority of conversations happening in a human language. That's the last point I'll make though, so you can keep it if you're pretty dead-set on it. I'd imagine this is trivial enough that it's not really worth either of our time continuing to discuss it unless someone else brings it up.

> 			`	As you are making the walk you hear questions posed and rumours stated. Occasionally you hear words like "survivor" and "dangerous" thrown in amongst the babble.`

Author: What I will say is that, as someone who has spent a good bit of time in foreign countries, one person speaking your language in a crowd of another tongue will stand out amidst the babble really really clearly. Heck, someone speaking a second language I was barely competent at stood out in a crowd enough for me to catch what was said.
It's also possible that the Hai have heard the news through copied human broadcasts, so they could also be bringing up the human word as clarity for the translation or context in a sentence. This would be a not unreasonable reason for you to hear such words even amongst a conversation conducted principally in another language.
In any event, you're correct that it's fairly trivial, and largely for effect.

				goto main
			label karma
			`	As you are making the walk you hear questions posed and rumours stated. Occasionally you hear words like "honourable" and "reasonable" thrown in amongst the babble.`
			label main
			`	You are eventually led into a very comfortable office where you are received by a mature looking Hai with the apropriate regalia of a diplomat. She holds out a paw to greet you, and says, "My name is Sayari. I was the envoy to the human government, a century ago."`
			`	The present media storm is all the evidence you need to know the Republic has never mentioned the Hai, nor publicly acknowledged receiving an envoy from them. If they're in on the secret, then it makes sense that they're probably still trying to work out a suspression strategy and have been too busy to send an envoy quicker than yourself.`

> 			`	After a moment to let you ponder, she continues, "Our reports tell us that you are a well-known individual, respected both by your own government and abroad in the other human governments. Our reports also tell us that 'Hai' is quickly becoming a buzzword in your space. A few traders brought the news ahead of you."`

Author: I think it's important that the player is identified as respected for the story to make sense, otherwise this seems good.

> 			`	After a moment to let you ponder, she continues, "Our reports tell us that you are a well-known individual, respected both by your own government and abroad in the other human governments. Our reports also tell us that 'Hai' is quickly becoming a buzzword in your space. A few traders brought the news ahead of you."`

Reviewer 3: Suggestion has been updated; added `respected`.

			`	She pauses and looks at you seriously.`
			`	"We think it would be prudent for you to represent your people as our point of contact. Would you accept such a role?"`

> 			`	You give it a moment of thought.`

Author: I don't know about you, but I like to consider whether there are other reasonable choices before I agree to things, even if there's only one reasonable position.

> 			`	You give it a moment of thought.`

Reviewer 3: Fair point. Comment updated.

			choice
				`	"It certainly seems rather practical."`
			`	"Fantastic," she says, though her tone is much more business like than the wording would imply, "In that case, if you could please take this to Earth and communicate with your various governments we can begin a dialogue on how best to deal with this current scenario."`
			choice

> 				`	"Various? We only have two governments."`

Author: One of the things with Derpy's checkmate re-write that I'd like to get in is the independence one way or another.

> 				`	"Various? We only have two governments."`

Reviewer 3: I think the improving of Checkmate isn't really being worked on right now, however. I'm not a fan of making this dependent on an idea that might or might not get into a PR that's currently a draft, unless you're planning on waiting until it's done to get this merged.

> 				`	"Various? We only have two governments."`

Author: In this case, Derpy has informed me this is unlikely to be merged before that happens anyway.

			`	"Oh," she says, "we've treated the Syndicate as a de facto government for many decades now; with the amount of influence they have it was the only prudent thing to do."`
			`	You take the boxed item with you and an escort awaits to return you to your ship. As the door behind you closes you are sure you hear Sayari mutter to someone, "It was really much easier when they just had one government though, what a mess."`
				accept
	
	
mission "Hai Leaks Response 2"
	landing
	name "Collect Representatives"
	description "Collect human government representatives from <stopovers> and take them to <destination>."
	source "Earth"
	stopover "Deep"
	stopover "Foundry"
	destination "Hai-home"
	blocked "You have a mission here for the Hai, but you need three bunks free."
	clearance
	to offer
		has "Hai Leaks Response 1: done"
	passengers 3
	on offer
		event "pirates respond to leaks"

> 		event "hai ladybug available"

Author: Oh yeah, forgot about that. This part got written so far before the ship got created it totally slipped my mind.

> 		event "hai ladybug available"

Reviewer 3: It's understandable. This is a hefty sum of work.

> 		log "People" "Xilin Yang" `Deputy Minister Yang works in the Foreign Affairs department of the Republic government. He is a pleasant and astute man with a firm handshake. Handling the Hai secrecy crisis has become his job while the majority of the foreign ministry remains engaged with the fallout from the Free Worlds and the recent peace.`

Author: This is, I think, an area where I disagree somewhat with the strict dedication to duplicating logbook structure. There are occasional references to a characteristic in some logs and I'm in favour of adding slightly more rather than strictly avoiding them.

> 		log "People" "Xilin Yang" `Deputy Minister Yang works in the Foreign Affairs department of the Republic government. He is a pleasant and astute man with a firm handshake. Handling the Hai secrecy crisis has become his job while the majority of the foreign ministry remains engaged with the fallout from the Free Worlds and the recent peace.`

Reviewer 3: While a fair point, I'd still say the amount of characterizing is a little too much. Maybe you could just remove `with a firm handshake`? If you prefer to keep it all in, then we'd be at an impasse, I guess. In that case, maybe we could wait for a third party with more experience than we have to weigh in? That might be making this a larger issue than it really is though. Not really sure. Your choice, I guess.

> 		log "People" "Xilin Yang" `Deputy Minister Yang works in the Foreign Affairs department of the Republic government. He is a pleasant and astute man with a firm handshake. Handling the Hai secrecy crisis has become his job while the majority of the foreign ministry remains engaged with the fallout from the Free Worlds and the recent peace.`

Author: I'm alright with taking that bit out.

> 		log "People" "Xilin Yang" `Deputy Minister Yang works in the Foreign Affairs department of the Republic government. He is a pleasant and astute man with a firm handshake. Handling the Hai secrecy crisis has become his job while the majority of the foreign ministry remains engaged with the fallout from the Free Worlds and the recent peace.`

Reviewer 3: Updated the suggestion according to the discussion.

		conversation
			`On approach to the spaceport on Earth you communicate your need to speak to an official of significant standing on diplomatic business. Since you are not a diplomat this does cause some minor kerfuffle in the process, but thanks to your renown from the Pug war you are able to blow past such petty objections sufficiently so as to shortcut the process. The diplomatic ID from the box Sayari gave you doesn't hurt either.`
			`	The Republic Deputy Minister for Foreign Affairs meets you in a hired, private room at the spaceport.`
			`	"Captain <last>," he steps around the table to grasp your hand in a firm handshake, "Xilin Yang, it's an honour to meet a war hero like yourself."`

> 			choice

Author: Yang's perspective doesn't change, but he's much too much the diplomat to let you know that. His position obliges him to appear to agree with you one way or another.

> 			choice

Reviewer 3: Ah, I see. Fair point; feel free to resolve the comment then.

> 				`	"It was a brutal business, but someone had to do it. For the people."`

Author: These choices are diametrically opposing attitudes, and they determine Xilin's predisposition towards the player. In this particular case the style is deliberate because I want the choice to slap the player with a "hey this choice as to how you represent yourseelf might mean something" warning.

> 				`	"It was a brutal business, but someone had to do it. For the people."`

Reviewer 3: I see what you're saying, but is there a way to enforce that onto the player without a style that seems so... "over the top" for both choices, at least? I've updated the suggestion below to try to keep the direct opposedness but give players at least one way to move through the conversation without being so dramatic about their wording.

> 				`	"It was a brutal business, but someone had to do it. For the people."`

Author: These new versions work well enough. :)

				`	"Please, it was far less dramatic than it's made out to be."`
					goto humble

> 			`	You play the part and do your best attempt at looking noble. Mr Yang seems more than happy to embody respect for the hero of his preferred narrative.`

Author: In this case it's actually *not* his preferred narrative, so he is embodying the act to suggest it is for your benefit. 
If he did agree he would personify it.

> 			`	You play the part and do your best attempt at looking noble. Mr Yang seems more than happy to embody respect for the hero of his preferred narrative.`

Author: They're both from backgrounds and careers with highly formal and structured settings but with no other titles, and none of the camaraderie that comes from fighting together. The choice to identify them as such was very deliberate.

> 			`	You play the part and do your best attempt at looking noble. Mr Yang seems more than happy to embody respect for the hero of his preferred narrative.`

Reviewer 3: I see your point. I'm still not really a fan because it still seems to stick out so much from the rest of the game to me, but in the end it's your choice, and there was intentionality behind it.

			`	"You're every inch what I expected," he assures you before moving to take his seat.`
				goto sit
			label humble

> 			`	You spread your hands in an appeasing gesture and explain, "You should listen to the xenoanthropologists, the Pug really were just here to test us, to make us really consider what mattered to our species."`

Author: There are only so many reasonable explanations for the actions of the Pug. If the player is a perceptive individual (implied by the choice that leads here), this kinda is the actual answer though. (Even if we don't know their motivations for why they wanted us to learn that lesson.)

> 			`	For a moment his expression falters and then he seems almost relieved, "You may be a better man than I expected to meet."`

Author: Nope, that's just him being diplomatic enough to fool you if you're a dumbass.

			`	He offers you a genuine smile as he takes his seat.`
			label sit
			`	You take your seat opposite him and you get down to business. You explain how the Hai have proposed that you be their official contact point for humanity on account of your fame from the war.`
			`	He asks a few questions for clarity and when you're done he requests if you'd kindly wait for a moment while he makes a call. It is a lengthy wait, and at one point an aide comes in and offers you refreshments while deliberations continue elsewhere. Much to your surprise the fruity beverage is actually quite light and not too sweet, reminding you briefly that this is actually the world where your major foods originally evolved.`
			`	Eventually Mr Yang returns, "So sorry for that, I hope you were looked after suitably while you waited."`

> 			`	Once you nod your assent he continues, "Well, obviously the Republic knows about the Hai, but for the moment our official line is that the government does not trade in rumours. However, clearly an appropriate response is needed. I have been informed that Republic Intelligence has gone through official channels to brief the Free Worlds Council just yesterday. They have demanded that they have a trained diplomatic representative along for any decisions. This is already standard practice for the Syndicate. If you are willing, I will come with you, and we shall meet two individuals known as Katya and Alastair on Deep and Foundry respectively. I believe you already know them?"`

Author: Another thing that I think should be included in Derpy's checkmate re-write (which he's said must be finished before this gets merged). I can put a branch in here though if it doesn't happen.

> 			`	Once you nod your assent he continues, "Well, obviously the Republic knows about the Hai, but for the moment our official line is that the government does not trade in rumours. However, clearly an appropriate response is needed. I have been informed that Republic Intelligence has gone through official channels to brief the Free Worlds Council just yesterday. They have demanded that they have a trained diplomatic representative along for any decisions. This is already standard practice for the Syndicate. If you are willing, I will come with you, and we shall meet two individuals known as Katya and Alastair on Deep and Foundry respectively. I believe you already know them?"`

Reviewer 9: Free Worlds Council should probably be substituted with Free Worlds Senate; whether the Coalition of Free Worlds is an autonomous region of the Republic (Checkmate) or an independent government (Reconciliation) the Council is dissolved after the end of the war per FW Epilogue: Alondo.

> 			`	Once you nod your assent he continues, "Well, obviously the Republic knows about the Hai, but for the moment our official line is that the government does not trade in rumours. However, clearly an appropriate response is needed. I have been informed that Republic Intelligence has gone through official channels to brief the Free Worlds Council just yesterday. They have demanded that they have a trained diplomatic representative along for any decisions. This is already standard practice for the Syndicate. If you are willing, I will come with you, and we shall meet two individuals known as Katya and Alastair on Deep and Foundry respectively. I believe you already know them?"`

Author: LordInsane is correct it should be Senate, but it's meant to imply that things are proceeding quickly. The people in authority have been contacted, but exactly who you're taking is still to be determined while you take however many days to fly across space and rendezvous with them.

> 			`	Once you nod your assent he continues, "Well, obviously the Republic knows about the Hai, but for the moment our official line is that the government does not trade in rumours. However, clearly an appropriate response is needed. I have been informed that Republic Intelligence has gone through official channels to brief the Free Worlds Council just yesterday. They have demanded that they have a trained diplomatic representative along for any decisions. This is already standard practice for the Syndicate. If you are willing, I will come with you, and we shall meet two individuals known as Katya and Alastair on Deep and Foundry respectively. I believe you already know them?"`

Reviewer 3: In that case, maybe you could change the dialogue to better convey that it's still "up in the air"? It's said with some decisiveness here that the individuals would be who Yang thinks, and Yang would have no reason to hide that it's still being decided to the player.

			`	You indicate as such.`

> 			`	"Excellent, in that case we should go to Deep first for speed, shall we be off?"`

Author: It's ok, I fixed that. You *should* go to Deep first for speed, but if you choose to ignore that it doesn't matter.

				accept
	on visit
		dialog phrase "generic stopover on visit"
	on complete

> 		log `You have brought a number of diplomatic representatives from each of the main human factions to Hai-home to attend to the crisis in a collaborative manner. They do not believe any of their automatic responses that have already been engaged will be enough to suppress the information leak, though there is a reasonable model for how long they have to work with. While they work on the problem though you have been asked to help route some pirates. Apparently less scrupulous individuals have reacted swiftly to the leak and investigated in the hopes of scoring quickly.`

Author: This feels excessively cut down.

> 		log `You have brought a number of diplomatic representatives from each of the main human factions to Hai-home to attend to the crisis in a collaborative manner. They do not believe any of their automatic responses that have already been engaged will be enough to suppress the information leak, though there is a reasonable model for how long they have to work with. While they work on the problem though you have been asked to help route some pirates. Apparently less scrupulous individuals have reacted swiftly to the leak and investigated in the hopes of scoring quickly.`

Reviewer 3: Was there anything you thought the original noted that was important information that mine didn't? I've added the bit back in about each representative being from each of the human factions, but besides that I didn't pick up on things that the original had that mine didn't sufficiently include.

> 		log `You have brought a number of diplomatic representatives from each of the main human factions to Hai-home to attend to the crisis in a collaborative manner. They do not believe any of their automatic responses that have already been engaged will be enough to suppress the information leak, though there is a reasonable model for how long they have to work with. While they work on the problem though you have been asked to help route some pirates. Apparently less scrupulous individuals have reacted swiftly to the leak and investigated in the hopes of scoring quickly.`

Author: I think the log being definitve about being "too late to suppress the leak" is excessively strong when the point of this bit is to find a way to buy time while they work out if it actually is too late or not. Plus, routing the pirates that have responded to it is part of the suppression, not merely assisting with law and order.

> 		log `You have brought a number of diplomatic representatives from each of the main human factions to Hai-home to attend to the crisis in a collaborative manner. They do not believe any of their automatic responses that have already been engaged will be enough to suppress the information leak, though there is a reasonable model for how long they have to work with. While they work on the problem though you have been asked to help route some pirates. Apparently less scrupulous individuals have reacted swiftly to the leak and investigated in the hopes of scoring quickly.`

Reviewer 3: I've tried to remove some of the definitiveness with which the leak getting out is addressed. I'm not sure what the best way to word your second point is, but I do think the log entry was too long as-is. Do you have any ideas on wording that might help us find a middle gorund?

		conversation

> 			`You are directed to an official landing pad on your return to Hai-home and it is not long before you are introducing your entourage to Sayari. Alondo in particular seems quite stoked to be here and seems to have a knack for charming Sayari with polite and insightful, if rather simple, questions throughout the diplomatic niceties.`

Reviewer 3: [Stoked](https://www.merriam-webster.com/dictionary/stoked) is noted specifically as slang, which was my reasoning behind this. Still up to you, but I thought I'd at least elaborate since you didn't seem to be a fan.

			`	You get down to business at the moment Sayari inquires how your journey went.`
			choice
				`	"I don't recall there being nearly so many pirates."`

> 			`	Sayari frowns and exhales a long breath through her nose, "Yes. Well it seems that the rumours of us, and the information making its way through your media has attracted every opportunist this side of the Deep."`

Author: One can sigh quickly, or one can sigh slowly. One can sigh sadly, longingly, with relief or with frustration.
The description is physical shorthand for slowly and with frustration.

> 			`	"Of course," she says, "some of the pirates were aware of us, and we were aware of them being aware of us. We had a quiet arrangement with a select few where we rewarded them for keeping the secret well contained within their own circles."`

Author: Because what follows is a qualifying statement I wanted to give it a bit of distance stylistically from the previous one and this seemed the smoothest way to do that. Were you just removing it for brevity?

> 			`	"Of course," she says, "some of the pirates were aware of us, and we were aware of them being aware of us. We had a quiet arrangement with a select few where we rewarded them for keeping the secret well contained within their own circles."`

Reviewer 3: Yes, that was my intent. But your point is a good reason to leave it in, so feel free to resolve.

			`	Alondo looks surprised, but Mr Yang and Ms Adrianopolous both merely nod sagely.`
			`	"Extra security forces are being called in, but we've been busier in Wah Ki than usual lately."`
			`	Mr Yang pulls out a datapad, "Well then I'm afraid I may have some bad news. May I?"`

> 			`	He gestures to a display and then takes control when indicated.`

Author: Good point, still though, ship stuff works easily enough. Plus, the Hai have been in contact and trading with us long enough to have got some compatibility protocols.

			`	"One of the things I brought with me was Republic Intelligence's estimate of the effectiveness of various information containment measures."`
			`	A series of charts are thrown up on the screen. You're not an expert at reading this sort of data but highlighted percentages ranging from as low as 4% to only as high as 39% do not look promising. The final graphic is a map of stages of what is labelled as 'artifact penetration'.`
			`	"Artifact penetration is the point at which a government cannot reasonably attempt to pretend something is other than the truth, at least not while still holding to the claim of being a representative government," explains Mr Yang.`
			`	He checks the date, "Our most optimistic estimates suggest the Republic will be forced to acknowledge the existence of the Hai within three months or lose so much credibility as to compromise electoral stability, the actual time limit is probably sooner."`
			choice
				`	"Would that be so bad?"`

> 			`	"It is always bad to be forced into acknowledging a secret, if we act sooner we may control the manner of that secret's release," says Yang.`

Author: It is worth noting that this is covered during the first contact with the Hai if the player bothers to ask questions. While it might be a bit of a reminder if people found the Hai long before this, it would be exceptionally repetitive if the player has followed the 'first time' route and then they brought it up again this swiftly.

			`	Sayari looks at you, "We will have to come up with a plan, and quickly, but for the moment may I ask a favour of the war hero in the room? Please meet up with the security forces converging on Allhome and help them route the pirates back to the other side of the wormhole. It will buy us some breathing space and some options."`

> 			`	It is so hard to say "no" to a Hai and their cute squirrelly features.`

Author: It's for juxtaposing against later.

			choice
				`	"I'll do what I can."`
					decline

#If a republic/syndicate campaign gets added this on complete text will need variants.
	
	
event "pirates respond to leaks"
	#Add pirate spawns to Waypoint and Heia Due
	system "Waypoint"
		government "Pirate"
		add fleet "Large Northern Pirates" 3000
		add fleet "Small Northern Pirates" 1000
		add fleet "Large Core Pirates" 3000
		add fleet "Small Core Pirates" 1000
		remove fleet "Small Hai"
		remove fleet "Large Hai"
	system "Heia Due"
		add fleet "Marauder fleet IX" 3000
		add fleet "Large Northern Pirates" 3000
		add fleet "Large Core Pirates" 3000
		add fleet "Anti-Pirate Hai (Wormhole Travel)" 1000
		
	
#If a republic/syndicate campaign gets added this text will need variants.
mission "Hai Leaks Response 2A"
	landing
	invisible
	source Deep
	to offer
		has "Hai Leaks Response 2: active"
	on offer
		log "People" "Alondo Gruyere" `Alondo is working to develop the diplomatic core for the Free Worlds. Due to a lack of serious appointed positions (with the few already existing occupied as ambassadors to the Republic), he is looking after the Free World's interests during this crisis.`
		conversation

> 			`When you land on Deep Katya is there waiting for you along with Alondo, "<first> it's so good to see you again! When the Republic briefed the Chancellor, he brought us in and we were very concerned by the lack of ability to reach you, but of course you would somehow manage to be in the thick of it before we even knew what was going on."`

Author: Wow I really did overuse that format enough that it became ingrained in your expectations. You're correct though.

> 			`When you land on Deep Katya is there waiting for you along with Alondo, "<first>! It's so good to see you again! When the Republic briefed the Chancellor, he brought us in and we were very concerned by the lack of ability to reach you, but of course you would somehow manage to be in the thick of it before we even knew what was going on."`

Author: These titles are never referred to yet at any point. This story introduces them officially for the first time.

> 			`When you land on Deep Katya is there waiting for you along with Alondo, "<first>! It's so good to see you again! When the Republic briefed the Chancellor, he brought us in and we were very concerned by the lack of ability to reach you, but of course you would somehow manage to be in the thick of it before we even knew what was going on."`

Reviewer 3: My concern here is that a Chancellor is never even mentioned during the entirety of the Free World's storyline, which specifically involves interacting with the Republic gov. If there was a Chancellor, it seems shocking that we would have never heard it even mentioned during Free Worlds. If the Chancellor is so integral to the story here that it doesn't work without their existence. I'd argue it needs to be written into the existing FW story in at least some form or fashion to avoid it feeling like a retcon.

> 			`When you land on Deep Katya is there waiting for you along with Alondo, "<first>! It's so good to see you again! When the Republic briefed the Chancellor, he brought us in and we were very concerned by the lack of ability to reach you, but of course you would somehow manage to be in the thick of it before we even knew what was going on."`

Author: I would counter that, unless executive power is vested in the head role (like with the President of the US), it's unlikely that any of the actual *work* involved in the FW plot would involve that leader on either side, over and above the roles of the government or the war ministry in general. The FW equivalent also would likely not have been elected until after the FW story and the council's disbandment.
Whereas a diplomatic crisis with a wholly foreign entity is likely going to significantly revolve around a face character in the media, even if the work still largely happens behind the scenes. In FW your involvement is with the active leadership (the council) and is focused on the real work. In this you're thrust into the limelight of media management.

			`	You hug them both in turn and return the greeting as you look from one to the other, "I thought it seemed odd when they said I was meeting you for this Katya, but it makes much more sense if you were just easier to get a hold of than Alondo here."`
			`	"Oh it was always going to be Alondo," she says, "but there's never a bad excuse for travelling with an old friend, especially when you're meeting other friends too."`
			`	"Do we have time for a drink before we must rush off?" asks Alondo.`
			`	You look around at the busy spaceport, no one has even begun looking at your ship for refueling.`
			`	"I think we can spare a short while, and you really should meet Mr Yang."`

> 			`	Mr Yang had been studiously pretending he wasn't eavesdropping from a distance of a few metres but, before you could call him over, Alondo's slightly suggestive comment of, "Oh I should definitely meet Mr Yang," had evidently been overheard.`

Author: In the FW campaign you find out Alondo is gay. I really just wanted Xilin to be a good guy whose character is not in doubt (with the way things happen later) so having Alondo find him attractive and that not be an issue seemed a good way of establishing a raport between them for later.

> 			`	Mr Yang had been studiously pretending he wasn't eavesdropping from a distance of a few metres but, before you could call him over, Alondo's slightly suggestive comment of, "Oh I should definitely meet Mr Yang," had evidently been overheard.`

Reviewer 3: That makes sense. I don't know whether it's just me that wouldn't pick up on that. If other people aren't going to pick up on that, maybe it should be reworded? My head was more thinking along the lines of Alondo had some sort of problem with Mr. Yang, which is why I thought it was odd.

> 			`	Mr Yang had been studiously pretending he wasn't eavesdropping from a distance of a few metres but, before you could call him over, Alondo's slightly suggestive comment of, "Oh I should definitely meet Mr Yang," had evidently been overheard.`

Author: I've had a think and I think I like this one being a bit obscure. It's insignificant enough that most people can brush it off if they're unsure, and then it's just a little easter egg for anyone who was paying attention to the right details.

				decline
	
mission "Hai Leaks Response 2B"
	landing
	source
		government "Republic" "Syndicate" "Free Worlds"
		not planet
			Deep
			Foundry
	to offer
		has "Hai Leaks Response 2: active"
	on offer
		conversation
			`While refuelling on <origin> you see that the media circus is continuing uabated. Apparently pirates have started making renewed attacks on trade lanes in the north, utilising some of the strange weapons observed in the recorded clip of Hai ships fighting each other. Goodness knows how they got their hands on them, but it can't be good news.`
				decline

mission "Hai Leaks Response 2C"
	landing
	invisible
	source Foundry
	to offer
		has "Hai Leaks Response 2: active"
	on offer
		log "People" "Terry Adrianopolous" `Protege to the Syndicate CEO, Terry Adrianopolous has been nominated by her mentor Alastair Korban to represent the Syndicate's interests in the crisis of the Hai's secrecy. She is a very capable woman who appears well equipped for climbing the corporate ladder, including an ability to act with apparently genuine friendliness to people whom she may consider her opponents.`
		conversation

> 			`Approaching Foundry you are hailed on a private encrypted channel and directed to a private hangar where you are greeted personally by CEO Alastair Korban.`

Author: I intended to add that, potentially I clobbered it accidentally.

			`	He smiles warmly, "Apologies Captain <last>, do excuse the cloak and dagger, but in the current media climate you and I are both too well known to be seen meeting without causing the rumour mill to grind out a new story."`
			`	"Now, allow me to introduce you to my protege Terry Adrianopolous, she will be accompanying you as the Syndicate representative."`

> 			`	The young woman goes from calculating to gregarious in the blink of an eye as your attention shifts to her. She's pleasant and approachable, and very articulate, but from your brief glimpse beforehand you have no doubt she's cut from the sort of cloth that makes her future CEO material. You're surprised that a woman would have made it this far in the Syndicate, but with your observation you resolve not to get on her bad side if at all possible.`

Author: I could, if I could work out how to write dialogue that portrayed what I was trying to get across.

> 			`	The young woman goes from calculating to gregarious in the blink of an eye as your attention shifts to her. She's pleasant and approachable, and very articulate, but from your brief glimpse beforehand you have no doubt she's cut from the sort of cloth that makes her future CEO material. You're surprised that a woman would have made it this far in the Syndicate, but with your observation you resolve not to get on her bad side if at all possible.`

Reviewer 3: It's a struggle. Maybe you could just have some normal dialogue that sounds very professional and note outside the dialogue your her characterization? That would do it for me.

> 			`	"Oh by the way," says Alastair, pausing briefly as he goes to depart, "The Navy and my security forces are working jointly to get some of our message relays up in the Far North to link through Ultima Thule. The pirates are wont to destroy them whenever they find them but we should be able to rely on at least basic communication to get through if you need it. Rather than flying back and forth for literally every question. That's all."`

Author: Alastair is an uncommon guy with a high degree of education behind him. It is both accurate and concise, and indicative of such.
I wouldn't use it in narration, but I won't dumb the text down in speech for people.

			branch delayedretire
				not "free worlds reconciliation"

> 			`	He leans in close to you and says quietly, "I know I said I was retiring, Terry convinced me to clean up my mess first. I am atoning for my failures, do look after her, she's a very loyal colleague."`

Author: He's within handshaking distance, and leaning in as he goes to depart? If you have a suggestion to make that clearer by all means.

> 			`	He leans in close to you and says quietly, "I know I said I was retiring, Terry convinced me to clean up my mess first. I am atoning for my failures, do look after her, she's a very loyal colleague."`

Reviewer 3: I've updated my suggestion for line 636 to try and accommodate this: instead of `goes to depart`, `turns to leave`. Also, it's never noted that he shakes your hand, which could also lead to the confusion I had.

			label delayedretire

> 			`	After which he finally, actually, departs for good.`

Author: It just felt long when I was writing it. XD

				decline

#This gets picked up silently on Hai-home when landing for response 2 - leads to go kill pirates mission with chat on Allhome.

> mission "Hai Leaks Response 3"

Author: That is commonly the format yes, but I wanted to keep the parts that will need changing when other campaigns get added in the same spot. It also gives more freedom to expand events in 3 if needed. I can promise it plays fine.

	name "Meet with Security Forces on Allhome"
	description "Meet with the Security Forces on Allhome before fighting the pirates."
	landing
	source "Hai-home"
	destination "Allhome"
	to offer
		has "Hai Leaks Response 2: done"

#Go kill pirates mission with chat on Allhome.
mission "Hai Leaks Response 4"
	name "Defeat the Black Lich"
	description "Defeat Black Lich's Marauder Leviathan at Waypoint."
	autosave
	landing
	source "Allhome"
	waypoint "Waypoint"

> 	destination "Allhome"

Author: It's necessary if you also have a waypoint.

	to offer
		has "Hai Leaks Response 3: done"
	on offer
		conversation
			`When you land on Allhome you are met by the Hai Captain of the local forces. You're not an expert at judging Hai ages but he appears to be a little younger than expected.`
			`	"Captain <last>! I heard you were coming to assist."`
			choice
				`	"Time to route some pirates baby!"`
				`	"I'm here to reinforce you, what can we do?"`

> 			`	"Uh well, yes. I presume you saw the pirates on your way in? They've been appearing en masse and overhwelming security forces to raid technology and goods from the Hai traders and their security. Some of them have even been sneaking down onto the planet and capturing Hai, taking them away to goodness only knows where."`

Author: It's meant to suggest the guy is stressed and not necessarily structuring his sentences formally and deliberately. Good catch on the saying though.

			choice

> 				`	"I had heard about the kidnapping."`

Author: 'had' for emphasis.

> 				`	"I had heard about the kidnapping."`

Reviewer 3: That `had` is grammatically incorrect (I think, I always struggle with these). [See: Past Perfect vs. Present Perfect](https://www.espressoenglish.net/difference-between-present-perfect-and-past-perfect-in-english/#:~:text=Free%20Download%3A%20500%2B%20English%20Phrases,point%20is%20in%20the%20past.)
I've updated the suggestion to be grammatically correct but remove the contraction.

			`	He nods, "Unfortunately, it's taking a while for them to redistribute an appropriately resourced defence force thanks to the Unfettered."`

> 			`	"Even more unfortunately," he continues, "it seems that the overabundance of these pirates is a result of one particular individual's success. His ship, called the Black Lich, is hanging out in Waypoint with the stolen Geocoris that we hear might have been converted into some kind of semi-stable weapons platform and now be being used as a temporary base."`

Author: I'm not sure <waypoints> works in conversation? But also the system is called Waypoint.

> 			`	"Even more unfortunately," he continues, "it seems that the overabundance of these pirates is a result of one particular individual's success. His ship, called the Black Lich, is hanging out in Waypoint with the stolen Geocoris that we hear might have been converted into some kind of semi-stable weapons platform and now be being used as a temporary base."`

Reviewer 3: I think it should, but since we're sure this works, this is fine. I've updated the suggestion.

			`	"Fortunately this gives us an opportunity though. If the kidnapped Hai are still on board we might be able to rescue them, and if we can defeat the Black Lich we may stem the tide of opportunists that are swarming through the wormhole."`
			`	"What do you say?"`
			`	The poor Captain seems desperate.`
			choice

> 				`	"We shall bring them safely home."`

Reviewer 3: You might consider mentioning more explicitly, both in the mission description and this conversation, that the ship must be captured to free the prisoners.

					accept
	npc
		government "Hai"
		personality escort heroic
		fleet "Anti-Pirate Hai (No Wormhole Travel)" 5
	npc
		government "Pirate"
		personality staying disables plunders harvests
		system "Waypoint"
		fleet "Marauder fleet IX" 4
		fleet "Hired Guns" 1
	npc kill
		government "Pirate"
		personality staying marked target
		system "Waypoint"
		ship "Marauder Leviathan (Weapons)" "Black Lich"
	to complete
		or
			has "00 Hai Leaks Response 4B: declined"
			has "Hai Leaks Response 4A: failed"
	on complete
		payment 1000000
		event "waypoint pirates vanquished"
		log `Defeated a nasty marauder whose ship was the "Black Lich". He had managed to kidnap a number of Hai straight off the planet with a combination of trickery and speed. You were rewarded well for this particular piece of aid.`
		conversation

> 			`When you return the Hai Captain thanks you profusely, "Please, the Allhome government has authorised a reward of <payment> for routing the problem."`

Author: This is flagged in the message you receive upon successfully capturing the Geocoris, at present there is no fix except to follow the instruction given unless a certain PR gets finalised and merged.

> 			`When you return the Hai Captain thanks you profusely, "Please, the Allhome government has authorised a reward of <payment> for routing the problem."`

Author: This is flagged in the message you receive upon successfully capturing the Geocoris, at present there is no fix except to follow the instruction given unless a certain PR gets finalised and merged.

> 			`When you return the Hai Captain thanks you profusely, "Please, the Allhome government has authorised a reward of <payment> for routing the problem."`

Author: Related to #5412 which was closed in favour of pursuing a solution like #5017

			branch "saved geocoris"
				has "00 Hai Leaks Response 4B: declined"
			`	He sighs deeply, "It is a shame though that the kidnapped Hai could not be saved, but the other Hai captains tell me it was a mess out there."`

> 			`	He places one of his furry hands comfortingly on your shoulder, "I'm sure you did all you could."`

Author: This is Allhome though, with a significant human population.

> 			`	He places one of his furry hands comfortingly on your shoulder, "I'm sure you did all you could."`

Author: I have reworked this section more significantly.

				goto end
			label "saved geocoris"
			`	"Your rescue of our kidnapped people is deeply appreciated also. Lucky for you, the merchant who owned the Geocoris has already filed the insurance claim, so you get to keep the ship as a trophy."`

> 			`	"I do need to do a full security scan of it before I let it go though, just to complete all the incident paperwork, so if you have any business on world here then you should take your time."`

Reviewer 3: ```suggestion
			`	"I do need to do a full security scan of it to complete all the incident paperwork before I let it go, so if you have any business on-world here, please take your time."`
```

> 			`	"I do need to do a full security scan of it before I let it go though, just to complete all the incident paperwork, so if you have any business on world here then you should take your time."`

Reviewer 3: It's odd to imply that there will be some significant wait when the player can just immediately leave after this conversation. Maybe add a little on at the end of what you have noting that `<first>` waits for a few hours or something?

Author: It's more meant to be a "if you wanted to play with outfits and stuff, now is a good time" moment. I see your point though.

			label end

> 			`	"In any event, let us hope a more permanent solution can be devised soon. In the meantime, I'm sure they'll be waiting for you back on Hai-home before too long."`

Reviewer 3: ```suggestion
			`	"In any event, let's hope a more permanent solution can be devised soon. In the meantime, I'm sure they'll be waiting for you back on Hai-home."`
```

> 			`	With a final nod of thanks he departs to attend to business, leaving you free to wander the spaceport until they release your new ship.`

Reviewer 3: ```suggestion
			`	With a final nod of thanks, he departs to attend to his business, leaving you free to wander the spaceport until they release your new ship.`
```

	
	
event "waypoint pirates vanquished"
	#Update pirate and anti-pirate spawns in Waypoint and Heia Due
	system "Waypoint"
		government "Hai"
		add fleet "Anti-Pirate Hai (Wormhole Travel)" 1000
		remove fleet "Marauder fleet IX"
		remove fleet "Anti-Pirate Hai (No Wormhole Travel)"
		remove fleet "Large Northern Pirates"
		remove fleet "Large Core Pirates"
	system "Heia Due"
		remove fleet "Marauder fleet IX"
		remove fleet "Large Northern Pirates"
		remove fleet "Large Core Pirates"

# It is impossible to implement a "capture and return npc" mission. 
# Once the ship is captured, the NPC is gone and the mission is failed.
# The NPC has an outfit used to determine if you managed to get it back safely.
mission "Hai Leaks Response 4A"
	name "Recover the Geocoris"

> 	description "Recover the stolen Geocoris supporting the pirates in Waypoint, and return to Allhome."

Reviewer 3: ```suggestion
	description "Recover the stolen Geocoris supporting the pirates in Waypoint, and return to Allhome. (You will need to swap the Geocoris to your flagship, and depart and land again in order to progress the missions properly.)"
```
I'd recommend putting this note here as well just to be safe in case the player accidentally skips over some dialogue.

	landing
	source "Allhome"
	waypoint "Waypoint"
	destination "Allhome"
	to offer
		has "Hai Leaks Response 3: done"
	npc board kill
		government "Pirate"
		personality staying marked mute
		system "Waypoint"

> 		ship "Geodesic Geocoris (Phylactery)" "Phylactery"

Reviewer 3: Isn't the ship's name `Black Lich`?

Author: No that's the marauder.

> 		dialog `After successfully boarding the Geocoris and defeating its pirate crews you are greeted by the relieved captive Hai. You promise to escort their ship safely to Allhome. (You will need to swap the geocoris to your flag and depart & land again in order to progress the missions properly.)`

Reviewer 3: ```suggestion
		dialog `After successfully boarding the Geocoris and defeating its pirate crews, you are greeted by the relieved captive Hai. You promise to escort their ship safely to Allhome. (You will need to swap the Geocoris to your flagship, and depart and land again in order to progress the missions properly.)`
```

	on fail
		dialog `You have failed to recover the Geocoris. This is not critical, but does have long-term consequences. If you want the most complete ending, revert to the autosave or another earlier snapshot of the game.`

# Must be able to resolve earlier than the rest in this set.
#The Geodesic Geocoris may become available to purchase specially later on.
mission "00 Hai Leaks Response 4B"
	landing
	source "Allhome"
	on offer
		require "Failed Control System"

> 		log `Successfully rescued the kidnapped Hai and reclaimed the Geocoris, something is different about it and it may be a valuable prize.`

Reviewer 3: ```suggestion
		log `Successfully rescued the kidnapped Hai and reclaimed the Geocoris. Something is different about it.`
```

		event "returned geocoris" 365
		conversation

> 			`You have successfully rescued the kidnapped Hai and reclaimed the Geocoris, something is different about it...`

Reviewer 3: ```suggestion
			`You have successfully rescued the kidnapped Hai and reclaimed the Geocoris. Something is different about it...`
```

		"heat dissipation" 0.4
		"fuel capacity" 800
		"cargo space" 1003
		"outfit space" 555
		"weapon capacity" 250
		"engine capacity" 134
		"energy capacity" 5200
		"shield generation" 6.0
		"shield energy" 8.8
		"hull repair rate" 0.8
		"hull energy" 0.8
		"hull heat" 1.2
		weapon
			"blast radius" 400
			"shield damage" 4000
			"hull damage" 2000
			"hit force" 6000
	outfits
		"Chameleon Anti-Missile" 2
		"Quad Blaster Turret" 2
		"Pulse Turret" 2
		
		"Quantum Keystone"
		"Boulder Reactor" 2
		"Liquid Helium Cooler"
		"Security Station" 2
		"Pulse Rifle" 30

		"A525 Atomic Steering"
		`"Biroo" Atomic Thruster`
		"AR120 Reverse Thruster" 2
		Hyperdrive

	turret 0 -82.5 "Pulse Turret"
	turret -95 -17 "Chameleon Anti-Missile"
	turret 95 -17 "Chameleon Anti-Missile"
	turret -29 3 "Quad Blaster Turret"
	turret 29 3 "Quad Blaster Turret"
	turret 0 91.5 "Pulse Turret"
	engine -95 41.5 0.9
	engine 95 41.5 0.9
	engine 1 154.5 0.9
	leak "leak" 40 50
	leak "flame" 40 80
	leak "big leak" 70 30
	explode "tiny explosion" 50
	explode "small explosion" 50
	explode "medium explosion" 50
	explode "large explosion" 50
	explode "huge explosion" 20
	"final explode" "final explosion large"

> 	description "This modified Geocoris has identical cargo capacity but is modelled off one that was the result of an alien technology being merged with a hybrid Human-Hai operating system. It has additional shield emitters in a geodesic pattern over its main cargo wings and a crucial element of the ship's system core is not manufactured, but grown from inital samples taken from the first occurrence. This means that some systems are built-in and unable to be removed, and also that it is substantially more expensive than its regular version. It is; however, significantly more survivable."

Reviewer 3: ```suggestion
	description "This modified Geocoris has identical cargo capacity, but is modeled off one that was the result of an alien technology being merged with a hybrid Human-Hai operating system. It has additional shield emitters in a geodesic pattern over its main cargo wings, and a crucial element of the ship's system core is not manufactured, but grown from initial samples taken from the first occurrence. This means that some systems are built-in and unable to be removed, and also that it is substantially more expensive than its regular version. It is, however, significantly more survivable."
```
Added comma after `capacity` and `wings`
Fixed spelling on `modeled`
Changed semicolon at end for grammatical correctness
What's the `first occurrence` you mention here? I have no idea what it's referencing, so you may want to elaborate.

	
ship "Geodesic Geocoris" "Geodesic Geocoris (Phylactery)"
	outfits
		"Electron Turret" 3
		"Chameleon Anti-Missile" 2
		"AR120 Reverse Thruster" 2
		"Boulder Reactor"
		"Hai Ravine Batteries"
		"Outfits Expansion"
		"Large Heat Shunt"
		"Quantum Keystone"
		"Pulse Rifle" 14
		`"Benga" Atomic Thruster`
		`"Bufaer" Atomic Steering`
		"Jump Drive"
		Hyperdrive
		"Failed Control System"
	turret "Electron Turret"
	turret "Electron Turret"
	turret "Chameleon Anti-Missile"
	turret "Chameleon Anti-Missile"
	turret "Electron Turret"

mission "Hai Leaks Response 5"
	landing

> 	name "Talk to Hai authorities to <planet>"

Reviewer 3: ```suggestion
	name "Talk to Hai authorities on <planet>"
```

	description "Meet Hai authorities on <destination>."
	source "Allhome"
	clearance
	destination "Hai-home"
	to offer
		has "Hai Leaks Response 4: done"

mission "Hai Leaks Response 6"

> 	name "Investigate rumours of kidnapped Hai."

Reviewer 3: ```suggestion
	name "Investigate rumors of kidnapped Hai"
```

	description "It's a statistical oddity, but Sayari's friend thinks someone is kidnapping Hai. Check spaceports in nearby parts of Hai and human space for evidence of a Hai trafficking operation."
	autosave
	landing
	source "Hai-home"
	clearance
	to offer
		has "Hai Leaks Response 4: done"
	on offer
		event "secret mountaintop"
		event "role-play fleet deception"
		# Hai like you a lot now, and as a result, you have permanent access to Hai-home:
		"reputation: Hai" >?= 100

> 		log `The working solution to the secrecy crisis for now is to have the Hai post a number of ships on the other side of the wormhole to play-act as an isolated furry role-play community. This is only a short-term solution. While they are working on a longer-term solution Sayari has quietly asked you to look into a rumour about more kidnapped Hai. She does not consider it to be a particularly likely scenario, but her contacts insist and so she would like you to be sure there's no issues lurking while the diplomatic process is working.`

Reviewer 3: ```suggestion
		log `The working solution to the secrecy crisis for now is to have the Hai post a number of ships on the other side of the wormhole to play-act as an isolated furry role-play community. This is only a short-term solution. Asked by Sayari to look into a rumor about more kidnapped Hai. She does not consider it to be a particularly likely scenario, but her contacts insist.`
```

> 		conversation

Reviewer 3: There are several spots here that could use some paragraph combining.

> 		conversation

Reviewer 3: As outlandish and unrealistic that a group of professional diplomats would agree the best course of action is to pretend an entire alien race is just furries, it's equally hilarious. I love it.

> 			`When you return to Hai-home you are met by a group of relieved diplomats.`

Reviewer 3: ```suggestion
			`When you return to Hai-home, you are met by a group of relieved diplomats.`
```

> 			`	Of all of them Alondo seems to be the least tired and takes the time to let you know what's been going on, "Captain <last>! You've missed quite the palava. Still, I doubt you lost much by missing out on trying to explain to Sayari what a 'furry' is."`

Reviewer 3: ```suggestion
			`	Of all of them, Alondo seems to be the least tired and takes the time to let you know what's been going on. "Captain <last>! You've missed quite the discussion. Still, I doubt you lost much. We've spent four hours trying to explain to Sayari what a 'furry' is."`
```
Never heard of "palava", and the only definition I could find was on Urban Dictionary. Probably best to stick to words we're sure are in official dictionaries. Tried to reword that last bit to avoid the double usage of `miss`.

Author: Turns out it's speller "palaver"

Reviewer 3: Suggestion updated to include correct spelling.

			choice

> 				`	"Oh I don't know, could've been entertaining."`

Reviewer 3: ```suggestion
				`	"What? That could've been entertaining."`
```

> 				`	"Hmm, yes, no probably not."`

Reviewer 3: ```suggestion
				`	"Hmm. No, I probably didn't miss much."`
```

> 			`	He chuckles, "Anyway, I have a hunch that we're going to be needing you before too long, but for now they're putting some Hai patrols out on the other side of the wormhole with a couple of Navy observers on board to 'play the part', as it were, of an exclusive role-play community, just chilling out here and having fun."`

Reviewer 3: ```suggestion
			`	He chuckles. "Anyway, I have a hunch that we'll need you before too long, but for now they're putting some Hai patrols out on the other side of the wormhole with a couple of Navy observers on board to 'play the part', as it were, of an exclusive role-play community, just chilling out here and having fun. Basically, we're going to pretend the Hai are furries for now."`
```

Author: Did we reeeaaally have to say it out loud?

Reviewer 3: Understandable, have a great day (suggestion updated)

> 			`	"Come though," he says, "I hear you've been instrumental in driving off some opportunistic pirates and helping stem the information leak, I'll buy you a drink on the other's behalf."`

Reviewer 3: ```suggestion
			`	"I hear you've been instrumental in driving off some opportunistic pirates and helping stem the information leak. Come on, I'll buy you a drink on the others' behalf."`
```

> 			`	Some hours later you leave the fancy cocktail bar Alondo took you to and begin making your way back to your ship to sleep, your cabin by far seems to be the best place to not be bothered.`

Reviewer 3: ```suggestion
			`	Some hours later, you leave the fancy cocktail bar Alondo took you to and begin making your way back to your ship to sleep.`
```
Didn't really see the point of noting the cabin was the best place to sleep.

Author: So that you go off on your own to your ship where Sayari can intercept you en route.

Reviewer 3: But doesn't `begin making your way back to your ship to sleep` fulfill that?

Author: Slightly, but I suppose as an honoured guest it could seem slightly odd that you're not put up somewhere nicer. You deliberately choosing to return to the ship for reasons over and above it being your ship felt significant when I wrote it. Idk though, I can still take it out if you dislike it.

Reviewer 3: Up to you. I think this is a pretty subjective take.
```suggestion
			`	Some hours later, you leave the fancy cocktail bar Alondo took you to and begin making your way back to your ship to sleep. Your cabin by far seems to be the best place to not be bothered.`
```
Here's a new suggestion with just grammar edits if you'd prefer to keep your original wording.

> 			`	Part way there though a hooded figure suddenly falls in step beside you.`

Reviewer 3: ```suggestion
			`	Partway there, a hooded figure suddenly falls in step beside you.`
```

			choice
				`	(Keep walking.)`
				`	(Ask them who they are.)`
			`	"Sssshhhh!" says the voice at your elbow, "do not react, and do not be alarmed, it is merely I."`

> 			`	You look at the figure out of the corner of your eye and connect the whispered voice to the squirrelly muzzle poking out. To your surprise it is Sayari.`

Reviewer 3: ```suggestion
			`	You look at the figure out of the corner of your eye and connect the whispered voice to the squirrelly muzzle poking out. To your surprise, it is Sayari.`
```

> 			`	"I should not be coming to you with this, so you must promise to say nothing of this to anyone just yet."`

Reviewer 3: ```suggestion
			`	"I should not be coming to you with this, so you must promise to say nothing of of it to anyone just yet."`
```

Author: You added an extra "of".

			choice
				`	"Of course."`

> 			`	"A very sweet friend of mine, who is much too enthusiastic and perhaps led by the vagaries of youth but nonetheless quite astute, keeps harping on to me about this theory of hers that a smuggling ring is kidnapping Hai and transporting them into human space."`

Reviewer 3: ```suggestion
			`	"A friend of mine is concerned about this theory of hers that a smuggling ring is kidnapping Hai and transporting them into human space."`
```
This is one spot where I think your specific writing style is too far removed from the rest of the game's narration. No offense intended, as I do like the way you write, but I do think things need to be consistent with the rest of the game's narration. I'd recommend rewriting much of Sayari's dialogue in the rest of this conversation accordingly. As such, the only edits I make for the rest of this conversation are grammar/spelling/punctuation, not style. There'd be too much I'd recommend changing that I'd be basically rewriting all of it.

Author: Sayari doesn't really get a lot of dialogue that's not strictly professional in the rest of the game. When she's not acting strictly in her professional role I was aiming for a 'divorced, middle-aged mum' vibe in her speech for personal interactions.
I am open to opinions on alternative styles though.

Reviewer 3: Yeah, I think I see what you're going for. My main concern was with words like `vagaries`, `nonetheless`, and `harping`, because words of that type are pretty uncommon in American English. Maybe you could strike a balance between that vibe and keeping those types of words to a minimum? If you want me to suggest a rewrite since I'm the one who has concern with it I can, but if you'd rather tackle it yourself that's cool too. Your choice; let me know.

Author: Ah I see. If you'd like to attempt a re-write along those lines be my guest. Otherwise I'll see about it at some point.

Reviewer 3: School's ramped up a good bit, so for the mean time feel free to do it yourself. Whenever things start to die down I'll be happy to suggest some things if you haven't done a rewrite already.

Reviewer 10: This was a long time ago, but I don't particularly mind that style for Sayari's speech, I actually quite like that sentence, though it is a little long, as long as her speech is consistent with this as appropriate.

> 			`	She pauses a moment for you to absorb this, "Of course, a thorough investigation has occurred on our side and no evidence of such things has turned up, but..."`

Reviewer 3: ```suggestion
			`	"Of course, a thorough investigation has occurred on our side, and no evidence of such things has turned up, but..."`
```

> 			`	"Well, you see we cannot investigate your space openly or easily. I have paid it little mind, written it off as one of her eccentricities, but things are very tense right now. The Council of Elders wants to maintain our secrecy, the human diplomats are willing to accomodate us, but we sense their governments grow tired of maintaining the fiction. We are well aware that human intelligence services are highly competent, and motivated by many different factors. It worries me that one of your factions may not be wholly honest, and may have knowledge they intend to use as ammunition if things don't work out well for them."`

Reviewer 3: This explanation seems a little sidetracked. Sayari doesn't really provide a clear explanation as to why she's bringing up this issue of her eccentric friend _now_, in the middle of the negotiations.

Author: If they're going to 'come out' to the world they want to know they're dealing honestly.

Reviewer 3: Could you elaborate on what you mean? I don't really understand what you're saying.

Author: Say you're a diplomat, you represent a group that is essentially 'secret', and now it seems that you have to navigate through a process of representing them as they either unveil that secret or go to great lengths to preserve it.
There is no official information suggesting that there should be a problem with the people you're negotiating with, but you have a friend/contact who is respected and influential in their own circles that insists there is a problem. 

You are subsequently presented with the opportunity to deal with a single individual who appears apparently trustworthy and who has the means to look into it. Knowing that if it is indeed true it will impact what you're trying to do: Do you keep quiet and ignore this doubt that your friend keeps bringing up, or do you see if you can get this individual to help lay that concern to rest for good before it's too late?

Reviewer 3: Oh, now I understand. Thanks for the clarification. I would say some of this explanation should get worked into this conversation, but that's just personal preference. Your choice.

Author: I could, and I may. I do like to leave some things for the canny reader to spot for themselves though. ;)

			`	You have not had much exposure to this elected "Council of Elders", but it seems they must hold some kind of final authority over decisions.`
			choice
				`	"But what may I do about it?"`

> 			`	"You are well-known, and must be canny to have got where you are. A Hai in trouble might recognise you, or you might perceive them when others would not. It would be in the interests of both our people to be sure that there is no such issue. Would you look for me? For both our sakes?"`

Reviewer 3: ```suggestion
			`	"You are well-known, and must be canny to have got where you are. A Hai in trouble might recognize you, or you might perceive them when others would not. It would be in the interests of both our people to be sure that there is no such issue. Would you look for me? For both our sakes?"`
```

			branch "about that"
				"hai slave prereq" >= 1
			choice
				`	"Sure, I can have a look around."`
					accept	
			label "about that"
			`	"Well, about that," you say, "I think I might have already come across some evidence of this before."`

> 			`	Sayari's hood nearly slips off as she looks up at you with a horrified expression, "Surely not?!"`

Reviewer 3: ```suggestion
			`	Sayari's hood nearly slips off as she looks up at you with a horrified expression. "Surely not?!"`
```

> 			`	You rush to reassure her, "Nothing concrete, of course I shall go and have a look to be sure."`

Reviewer 3: ```suggestion
			`	You rush to reassure her. "Nothing concrete, of course. I shall go and have a look to be sure."`
```

				accept
	to complete
		"hai slave prereq" >= 3
	on complete
		log `You found proof of more kidnapped Hai. Sayari's fears appear to have been entirely justified...`

mission "Hai Leaks Response 7"
	landing
	invisible
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Hai"
		not attributes "uninhabited"
	to offer
		has "Hai Leaks Response 6: active"
		"hai slave prereq" >= 3
	on offer
		conversation

> 			`You think you have enough evidence of kidnapped Hai to be sure of the bad news you bring. There may be other Hai you could still save, but you have enough information for Sayari.`

Reviewer 3: ```suggestion
			`You think you have enough evidence of kidnapped Hai to be sure that something's going on here. There may be other Hai you could still save, but you should go tell Sayari first to see how to proceed.`
```

Author: I think it's pretty clear that it's bad news though. 
It was "Can you find evidence of this?"
"Yes."
"Well crap."

Reviewer 3: Suggestion updated.

	`	The host stresses the word 'had' so much his voice squeaks, but the audience's attention has certainly been gained.`
	`	"But first, let's take a look at a clip."`
	`	You raise an eyebrow with surprise.`
	branch "hai knowledge"
		has "First Contact: Hai: offered"
	`	A clip of alien space ships in combat with weapons quite unlike anything produced in human space is shown. You've seen things, but not this.`
	`	When it's finished the young man, who is very well presented but clearly has a bit of the paranoid about him that is plain to see even through a screen, is encouraged to talk about where he got the footage. He talks at length about an ongoing conflict that threatens to spill over and drag all of humanity back to war: a war waged by an alien species known as the 'Hai'.`
	`	This is, of course, ridiculous... but maybe it would be good to go investigate this for yourself. You listen in to collect the relevant details for future reference.`
		accept
	label "hai knowledge"
	`	A clip of alien space ships in combat with weapons quite unlike anything produced in human space is shown, and you recognize Shield Beetles hammering away at each other.`

> 	`	When it's finished the young man, who is very well presented but clearly has a bit of the paranoid about him that is plain to see even through a screen, is encouraged to talk about where he got the footage. He talks at length about an ongoing conflict that threatens to spill over and drag all of humanity back to war: a war waged by an alien species known as the 'Hai'.`

Author: ```suggestion
	`	When it's finished the young man, who is very well presented but is clearly anxious about being here, is encouraged to talk about where he got the footage. As he speaks he has a bit of a paranoid manner to him that is plain to see even through a screen, talking at length about an ongoing conflict that threatens to spill over and drag all of humanity back to war: a war waged by an alien species known as the 'Hai'.`
```

			`	"Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew kidnapped nearly four dozen Hai and hijacked a Geocoris before making off with their spoils."`
				goto "that's awful"
			label "skipped tours"
			`Soon after you land on <planet>, you notice a news broadcast on a nearby wall. "Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew kidnapped nearly four dozen Hai and hijacked a Geocoris before making off with their spoils."`

> 				goto "that's awful"

Author: Good catch, I'd have just been thinking, "this goes to, or continues, and this goes to or continues, and finally this just goes to..."

> 				goto "that's awful"

Author: ```suggestion

```

			label "that's awful"
			`	At a table nearby you hear a Hai staff member exclaim, "Oh, that's awful!"`
			`	The rumors must be spreading further by the day. There's no doubt even more problems than this marauder will follow.`

> 			`	This thought has no sooner crossed your mind when your solitude is interrupted by a small group of somewhat official-looking Hai.`

Author: Ok now that you point this out to me I actually hate this whole sentence. Something feels like it's ass-about-face. Do you reckon you could have a crack at re-doing it entirely?

> 			`	This thought has no sooner crossed your mind when your solitude is interrupted by a small group of somewhat official-looking Hai.`

Reviewer 11: ```suggestion
			`	No sooner has this thought crossed your mind than your solitude is interrupted by a small group of somewhat official-looking Hai.`
```

This would be the easiest general fix, just transplanting that phrase to the front.

	landing
	source "Hai-home"
	destination "Allhome"
	to offer
		has "Hai Leaks Response 2: done"

#Go kill pirates mission with chat on Allhome.
mission "Hai Leaks Response 4"
	name "Defeat the Black Lich"
	description "Defeat Black Lich's Marauder Leviathan at <waypoints>, then return to <origin>."
	autosave
	landing
	source "Allhome"
	waypoint "Waypoint"
	destination "Allhome"
	to offer
		has "Hai Leaks Response 3: done"
	on offer
		conversation
			`When you land on Allhome, you are greeted by the Hai Captain of the local forces. You're not an expert at judging Hai ages, but he appears to be a little younger than expected.`
			`	"Captain <last>! I heard you were coming to assist."`
			choice
				`	"Time to route some pirates baby!"`
				`	"I'm here to reinforce you. What can I do?"`
			`	"Uh well, yes. I presume you saw the pirates on your way in? They've been appearing en masse and overwhelming security forces to raid technology and goods from the Hai traders and their security. Some of them have even been sneaking down onto the planet, capturing Hai, and taking them away."`
			choice
				`	"I have heard about the kidnapping."`

> 			`	He nods. "Unfortunately, it's taking a while for them to redistribute an appropriately resourced defense force, thanks to the Unfettered. It's bad enough that we've had to bring the old Ladybug models out of mothballs to bolster our forces here."`

Author: You picked up the intent precisely.

event "hai ladybug available"
	shipyard "Hai Intermediate"
		"Ladybug"
	shipyard "Hai Advanced"
		"Ladybug"

mission "Hai Reveal: Reputation Reset"
	landing
	invisible
	to offer
		has "event: pug media coverage"
	on offer
		# Sync reputations for aliases of governments:
		"reputation: Deep Security that won't enter wormhole" = "reputation: Deep Security"
		"reputation: Republic that won't enter wormhole" = "reputation: Republic"
		"reputation: Free Worlds that won't enter wormhole" = "reputation: Free Worlds"
		"reputation: Syndicate that won't enter wormhole" = "reputation: Syndicate"

		# Make sure the new governments start with appropriate rep:
		"reputation: Human Alliance that won't enter wormhole" = 1000
		"reputation: Human Alliance" = 1000
		"reputation: Hai (Friendly Unfettered)" = 1000
		"reputation: Bounty Hunter that Attacks Hai" = -1000
		"reputation: Philosophers" = 1000
		"reputation: Pirate (Devil-Run Gang)" = -1000
		fail

mission "Event: Pug Media Coverage"
	landing
	source
		government "Republic" "Syndicate" "Free Worlds"
		attributes "spaceport"
	to offer
		has "event: pug media coverage"
	on offer
		event "hai secret leaks" 10 15
		conversation

> 			`As you walk through the spaceport, you can't help but notice that coverage on the events of the Pug invasion still continues to dominate headlines, even though it's been weeks now.`

Author: Sure, I agree - for the _player_ though the game pretty much goes straight back to normal after it's all done. This is meant to remind the player of exactly that which you're observing: _This stuff was important and is still being talked about._

			`	Your thoughts, you keep to yourself.`
				decline

event "hai secret leaks"

conversation "hai reveal free worlds news"
	`You are relaxing within the spaceport while your ship refuels after conducting a spot of business. On the view-screen is some talk show - not something you specifically pay attention to, but apparently quite a popular one out of Tarazed.`
	`	A young and fresh-faced man who looks incredibly ill at ease has just been brought out as a surprise guest by the host Jonathon Camembert.`
	`	"So now in our surprise segment, let me introduce you to Ronald Campbell. He once crewed on one of these big trade ships, but after seeing what he saw, he just had to come forward!"`
	`	The host stresses the word 'had' so much his voice squeaks, but the audience's attention has certainly been gained.`
	`	"But first, let's take a look at a clip."`
	`	You raise an eyebrow with surprise.`
	branch "hai knowledge"
		has "First Contact: Hai: offered"
	`	A clip of alien space ships in combat with weapons quite unlike anything produced in human space is shown. You've seen things, but not this.`

> 	`	When it's finished the young man, who is very well presented but is clearly anxious about being here, is encouraged to talk about where he got the footage. As he speaks he has a bit of a paranoid manner to him that is plain to see even through a screen, talking at length about an ongoing conflict that threatens to spill over and drag all of humanity back to war: a war waged by an alien species known as the 'Hai'.`

Author: Going back to the other comments on the mannerisms, I'm _again_ not sure this change to the paranoid bit hits the mark I was aiming for. The rest of it I'm good with tho

> 	`	This is, of course, ridiculous... but maybe it would be good to go investigate this for yourself. You listen in to collect the relevant details for future reference.`

Author: Specifically it's the idea that we've been living next to an alien race that we've all just agreed to keep secret for many years, that is somehow in a war about to spill over and threaten everyone without us knowing about it.

It's not that there's an alien race there, or that they're at war, or that they're secret. It's the combination of all of those together which seems (and frankly is reasonable to call) ridiculous.
And ultimately, it *doesn't* threaten to spill over. The Unfettered are entirely still contained at this point, and it's just a guy spilling a secret for 30 seconds of fame.

	`	When it's finished the young man, who is very well presented but is clearly anxious about being here, is encouraged to talk about where he got the footage. As he speaks he has a bit of a paranoid manner to him that is plain to see even through a screen, talking at length about an ongoing conflict that threatens to spill over and drag all of humanity back to war: a war waged by an alien species known as the 'Hai'.`
	`	You suspect it might be time to go visit your Hai friends again.`
		accept

conversation "hai reveal republic or syndicate news"
	`You are relaxing at the spaceport while your ship refuels when a breaking news broadcast comes on; it's not urgent enough to actually interrupt scheduled content, but urgent enough to fill the first ad break.`
	`	The severe-looking anchorwoman bores down the camera with an expression that demands attention.`
	`	"An ex-crewman employed by an itinerant trader has come forward to media in the Free Worlds about hostile aliens just beyond our borders. He claims to have brought video proof of an ongoing conflict that threatens to spill over and drag all of humanity back to war. Stay tuned for our full segment at six."`

> 	`	You raise your eyebrows with surprise.`

Reviewer 10: As above.

> 	`	That such news should come out so soon after the Pug smacks of a hoax. A quick glance at the clock shows you only have ten minutes to wait, so you stick around for the full segment out of curiosity.`

Reviewer 10: "smacks of a hoax" is unexpected phraseology from the ES narrator.

> 	`	Right on schedule the news returns with the severe anchorwoman. It turns out that the crewman in question had been picked up by a minor media outlet on Tarazed. He had apparently traveled many weeks before deciding to seek the media in order to avoid being shut down by Republic Intelligence if he contacted them elsewhere. The man is fairly young, and very well presented, though he has a bit of the paranoid about him that is plain to see even through a screen.`

Reviewer 10: ```suggestion
	`	Right on schedule the news returns with the severe anchorwoman. It turns out that the crewman in question had been picked up by a minor media outlet on Tarazed. He had apparently traveled many weeks before deciding to seek the media in order to avoid being shut down by Republic Intelligence if he contacted them elsewhere. The man is fairly young, and very well presented, though he displays paranoia that is plain to see, even through a screen.`
```

	branch "hai knowledge"
		has "First Contact: Hai: offered"

> 	`	A clip of alien space ships in combat with weapons quite unlike anything produced in human space is shown. You've seen things, but not this.`

Reviewer 10: As above.

Author: ```suggestion
	`	They show a short video featuring alien ships, not Pug, or Quarg, but something else. They are engaged in combat with each other. They fire weapons quite unlike anything produced in human space.`
```

		accept	
	label "hai knowledge"
	`	A clip of alien space ships in combat with weapons quite unlike anything produced in human space is shown, and you recognize Shield Beetles hammering away at each other.`
	`	It is only after this that the anchorwoman reveals that, while this was initially treated as a hoax, one of their own reporters posed a timely question to a well-respected trader and caused quite the stir when he slipped up and confirmed the existence of the aliens known as 'Hai'.`
	`	You suspect it might be time to go visit your Hai friends again.`
		accept

conversation "What is going on in Hai space? (Have not met Unfettered.)"
	`Landing in Hai space draws a much different type of attention than expected. Only recently you've begun to acclimate yourself to getting positive attention. As a war hero you're something of a minor celebrity now back in human space, but here suddenly mothers are keeping tight grips on their children, and passers-by look upon you with a hint of wariness.`

> 	`	You realize that the few humans who have settled here among the Hai are all wearing pink at the moment, and when you catch the eye of one lady who looks friendly you greet them to ask a few questions.`

Reviewer 10: Is the use of 'them' because we are greeting a group? If it's just the one person, you've already called her a "lady" nad go on to use feminine pronouns later. I'm not sure it makes much sense to use a gneder neutral one here.
```suggestion
	`	You realize that the few humans who have settled here among the Hai are all wearing pink at the moment, and when you catch the eye of one lady who looks friendly you greet her to ask a few questions.`
```

		add fleet "Marauder fleet IX" 2000
		add fleet "Anti-Pirate Hai (No Wormhole Travel)" 700

mission "Trader Directions to Hai Space"
	landing
	name "Investigate the Hai"
	description "Follow the coordinates given to you by Captain Casey and meet the Hai."
	source "Sunracer"
	waypoint "Ultima Thule"
	destination "Allhome"
	to offer
		or
			has "ZZ Event Hai Secret Leaks: RS: done"
			has "ZZ Event Hai Secret Leaks: FW: done"
	on offer
		conversation
			`Arriving at the only place you can think of where you feel sure you can find out more information, you find that <origin>, like most places, is abuzz with the recent news. Avoiding the many large, cheap, and boisterous establishments that service the many crews and workers of the Megaparsec corporate machinery here, you make your way to one of the marginally more up-market places, cunningly marketed as a merchant-only dive.`
			`	The endless cycle of both news and government propaganda displayed on most screens in major thoroughfares is remarkably absent here, with only one of the screens on display covering the "violent alien race" at all - running stats on how many different versions of the story are now being told.`

> 			`	Casting around the room, you pick out a merchant who looks like she's been around the block a few times. In addition to looking like she could pick up most people and break them over her knee, she's conspicuous by the threatening glare you watch her give a younger captain who left a table talking about the Hai to approach her... briefly. After the captain is briskly sent away, you make your way over. Her glare falters in a flash of recognition before subsiding to a reserved scowl.`

Author: It is definitely the correct use of `casting`,  the addition of a choice is probably worthwhile here though; good call.

> 			`	She waits until you're directly next to her table. "The famous Captain <last>. Which ancient Sumerian deity did I offend to be granted the gift of your attention, hey?"`

Author: You are possibly the most famous captain in known space at this point, at least for a while. Even if I thought you were a complete load of tripe, if you showed up at my table in person I'd give you an opportunity to at least explain yourself.

				accept

	#Goto Stop 3 Wah Yoot (Darkcloak) - must happen after Quarg first contact
mission "ZZ Hai Tour [3] Meet Unfettered"
	landing
	name "Meet the Rebels on <planet>"
	description "Go to <destination> and learn more about the Hai's conflict."
	source "Alta Hai"
	destination "Darkcloak"
	to offer
		has "Hai Tour [2] Alta Hai: done"
	on offer
		conversation
			`This is a complete Quarg ringworld, the product of millennia of construction effort which has consumed every other stellar object in this system in order to provide the raw materials. The inner edge of the ring is filled with solar panels, hydroponic gardens, living quarters, and sunlit common spaces. The outer edge contains hangars and docks for ships while housing storage facilities for cargo.`
			`	Pondering your questions, you find a Quarg and ask, "So this isn't Hai space then, but it's a dead end?"`
			`	The Quarg explains, "Much like in Enif in human space, this is where we live as friends to the Hai."`
			`	This explanation seems typically Quarg, but it's the best one you're able to get. Before you go, you ask, "Do you know about the rebels I've been hearing about?"`

> 			`	The Quarg responds, "The Hai and their rebels are the same. You should travel to Darkcloak in Wah Yoot to understand."`

Author: > Can you elaborate on this?

Intentionally, no.

> I'm not sure a Quarg would actually tell us to go there, especially without some warning that they will fire upon us, unless it wants us to be killed.

You did just fight off the Pug, the Quarg know this. You should be able to handle a brush with the Unfettered.

			`	It seems like there's nothing else for it but to see for yourself.`
				accept
	
	#Goto Stop 4 Wah Ki (Cloudfire)
	# Must happen before unfettered first contact
mission "00 Hai Tour [4] Cloudfire"
	landing
	name "Find out about the Rebels on Cloudfire"
	description "Go to Cloudfire and compare the Hai's side of the conflict."
	source "Darkcloak" 
	destination "Cloudfire"
	to offer
		or
			has "Hai Witness Unfettered [1]: done"
			has "ZZ Hai Tour [3] Meet Unfettered: done"
	on offer
		conversation
			`You had to bribe your way through just to land without being blown up. These Unfettered Hai do not seem friendly, but it's time to find out what they're about.`
				accept
	on complete
		conversation

> 			`You dodge the conflict in space and come in to land. Cloudfire's atmosphere is unusually hazy, with almost constant cloud cover, making agriculture unproductive even though it is a warm and damp environment. There are several major cities here that were abandoned centuries ago, likely due to the conflict with the Unfettered, and the one major city that remains is the closest thing the Hai have to a military base, with new supplies of weapons and ammunition constantly pouring in from the rest of Hai space.`

Author: Take that up with the existing planet description.

But somewhere from 100-250 years ago.

			label main
			`	They leave without waiting for your agreement, but then, you did come here to see what was going on.`
				accept

#All prior branches merge here.
	
mission "Hai Leaks Response 1"
	landing
	name "Hai message to <planet>"
	description "Take the diplomatic ID from the box from Sayari to <destination>."
	source "Hai-home"
	destination "Earth"
	to offer
		or
			has "Hai Tour [5] Tour Ends: done"
			has "ZZ Event Hai Secret Leaks: RSHU: done"
			has "ZZ Event Hai Secret Leaks: FWHU: done"
	on offer
		log "People" "Sayari" `Ambassador Sayari, a Hai, served as their envoy to the human government a century ago (although no public records of this exist in human space). Her experience has placed her in charge of coordinating the Hai response to the crisis of their continued secrecy.`
		log `The Hai are very real, and human worlds seem poised to become aware of their existence. The Hai would like to avoid this. Agreed to assist by representing their interests where travel to human space is required. Their desire was for a human, yet somewhat neutral, party.`
		conversation

> 			`As you arrive on Hai-home, you find that a reasonable security detachment is waiting at your designated landing berth. The Hai aren't usually ones to be concerned about such things, but you suspect that, as an apparent "war hero", they may be more for your own safety than the Hai's.`

Author: Good catch - should track if you discovered the Hai as a result of this chain and use that for a branch.

			`	"Fantastic," she says, though her tone is much more business like than the wording would imply. "In that case, if you could please take this to Earth and communicate with your various governments we can begin a dialogue on how best to deal with this current scenario."`
			`	She hands you an officially branded box containing what appears to be a diplomatic ID, which appears to both identify you in human standard and, from context, in Hai.`
			choice
				`	"Various? We only have two governments."`
			`	"Oh," she says, "we've treated the Syndicate as a de facto government for many decades now; with the amount of influence they have it was the only prudent thing to do."`
			`	You take the boxed item with you, and an escort awaits to return you to your ship. As the door behind you closes, you hear Sayari mutter to someone, "It really was much easier when they just had one government though. What a mess."`
				accept
	
	
mission "Hai Leaks Response 2"
	landing
	name "Collect Representatives"
	description "Collect human government representatives from <stopovers> and take them to <destination>."
	source "Earth"
	stopover "Deep"
	stopover "Foundry"
	destination "Hai-home"

> 	blocked "You have a mission here for the Hai, but you need three bunks free."

Author: I would argue that it's extremely unlikely for the player to go from Hai Home to Earth at this stage of the game and *not* have 3 bunks free. I suppose it's just something of a meta reminder to people to get their affairs in order if they haven't.

			label delayedretire
			`	Alastair says goodbye and departs, leaving you with Terry.`
				decline

#This gets picked up silently on Hai-home when landing for response 2 - leads to go kill pirates mission with chat on Allhome.
#No longer gets picked up silently due to write-read order style preferences - but may need to be moved back to Response 2 or have multiple versions of this mission when other campaigns exist.
mission "Hai Leaks Response 3"
	name "Meet with Security Forces on Allhome"
	description "Meet with the Security Forces on <destination> to fight pirates."
	landing
	source "Hai-home"
	destination "Allhome"
	to offer
		has "Hai Leaks Response 2: done"
	on offer
		conversation
			`You are directed to an official landing pad on your return to Hai-home, and it is not long before you are introducing your entourage to Sayari. Alondo in particular seems excited to be here and seems to have a knack for charming Sayari with polite and insightful, if rather simple, questions throughout the diplomatic niceties.`
			`	You get down to business when Sayari inquires how your journey went.`
			choice

> 				`	"I don't recall there being nearly so many pirates before."`

Author: Yes

			`	Sayari frowns and exhales a long breath through her nose, "Yes. Well, it seems that the rumors making their way through your media have attracted every opportunist this side of the Deep."`
			`	"Of course," she says, "some of the pirates were aware of us, and we were aware of them being aware of us. We had a quiet arrangement with a select few where we rewarded them for keeping the secret well contained within their own circles."`
			`	Alondo looks surprised, but Mr. Yang and Ms. Adrianopolous both merely nod.`
			`	"Extra security forces are being called in, but we've been busier than usual in Wah Ki with our northern brethren."`
			`	Mr. Yang pulls out a datapad. "Well then, I'm afraid I may have some bad news. May I?"`
			`	He gestures to a display and then takes control when indicated. "One of the things I brought with me was Republic Intelligence's estimate of the effectiveness of various information containment measures."`

> 			`	A series of charts are thrown up on the screen. You're not an expert at data analysis, but highlighted percentages ranging from 4% to 39% do not look promising. The final graphic is a map of stages of what is labeled as 'artifact penetration'. "Artifact penetration is the point at which a government cannot reasonably attempt to pretend something is other than the truth, at least not while still holding to the claim of being a representative government," explains Mr. Yang.`

Author: I think I stole it from a sci-fi book actually.

			`	He checks the date. "Our most optimistic estimates suggest the Republic will be forced to acknowledge the existence of the Hai within three months, or we would lose so much credibility as to compromise electoral stability. The actual time limit is probably sooner."`
			choice
				`	"Would that be so bad?"`
			`	"It is always bad to be forced into acknowledging a secret. If we act sooner, we may control the manner of that secret's release," says Yang. "Plus, while the Hai have let the trickle of humans that find the wormhole settle here peacefully, a massive influx is a problem nobody really wants to have to deal with."`

> 			`	Sayari looks at you. "We will have to come up with a plan, and quickly, but for the moment may I ask a favor of the war hero in the room? Please meet up with the security forces converging on Allhome and help them route the pirates back to the other side of the wormhole. It will buy us some breathing room and options."`

Author: ```suggestion
			`	Sayari looks at you. "We will have to come up with a plan, and quickly, but for the moment may I ask a favor of the war hero in the room? Please meet up with the security forces converging on Allhome and help them rout the pirates back to the other side of the wormhole. It will buy us some breathing room and options."`
```
No. _squints at the revision history_

			`	It is so hard to say "no" to a Hai and their cute squirrelly features.`
			choice
				`	"I'll do what I can."`

> 					decline

Author: I believe this works in tandem with another/a hidden mission and should in fact be `decline`

	landing
	name "Talk to Hai authorities on <planet>"
	description "Meet Hai authorities on <destination>."
	source "Allhome"
	clearance
	destination "Hai-home"
	to offer
		has "Hai Leaks Response 4: done"

mission "Hai Leaks Response 6"
	name "Investigate rumors of kidnapped Hai"
	description "It's a statistical oddity, but Sayari's friend thinks someone is kidnapping Hai. Check spaceports in nearby parts of Hai and human space for evidence of a Hai trafficking operation."
	autosave
	landing
	source "Hai-home"
	clearance
	to offer
		has "Hai Leaks Response 4: done"
	on offer
		event "secret mountaintop"
		event "role-play fleet deception"
		event "slightly lost" 10
		# Hai like you a lot now, and as a result, you have permanent access to Hai-home:
		"reputation: Hai" >?= 100
		log `The working solution to the secrecy crisis for now is to have the Hai post a number of ships on the other side of the wormhole to play-act as an isolated furry role-play community. This is only a short-term solution. Asked by Sayari to look into a rumor about more kidnapped Hai. She does not consider it to be a particularly likely scenario, but her contacts insist.`
		conversation

> 			`When you return to Hai-home, you are met by a group of relieved diplomats. Of all of them, Alondo seems to be the least tired and takes the time to let you know what's been going on. "Captain <last>! You've missed quite the palaver. Still, I doubt you lost much. We've spent four hours trying to explain to Sayari what a 'furry' is."`

Reviewer 10: I've been gone for days or weeks. Four hours doesn't sound that long by comparison. Perhaps a day or two.

			choice
				`	"What? That could've been entertaining."`
				`	"Hmm. No, I probably didn't miss much."`
				`	"Oh, I could have told her all about it..."`

> 			`	He chuckles. "Anyway, I have a hunch that we'll need you before too long, but for now they're putting some Hai patrols out on the other side of the wormhole with a couple of Navy observers on board to 'play the part', as it were, of an exclusive role-play community, just chilling out here and having fun."`

Reviewer 10: ```suggestion
			`	He chuckles. "Anyway, I have a hunch that we'll need you again before too long, but for now they're putting some Hai patrols out on the other side of the wormhole with a couple of Navy observers on board to 'play the part', as it were, of an exclusive role-play community, just chilling out here and having fun."`
```

			`	"I hear you've been instrumental in driving off some opportunistic pirates and helping stem the information leak. Come on, I'll buy you a drink on the others' behalf."`
			`	Some hours later, you leave the fancy cocktail bar Alondo took you to and begin making your way back to your ship to sleep. Your cabin by far seems to be the best place to not be bothered.`
			`	Partway there, a hooded figure suddenly falls in step beside you.`
			choice
				`	(Keep walking.)`
				`	(Ask them who they are.)`
			`	"Sssshhhh!" says the voice at your elbow. "Do not react, and do not be alarmed, it is merely I."`
			`	You look at the figure out of the corner of your eye and connect the whispered voice to the squirrelly muzzle poking out. To your surprise, it is Sayari.`
			`	"I should not be coming to you with this, so you must promise to say nothing of it to anyone just yet," she says.`
			choice
				`	"Of course."`
			`	"A very sweet friend of mine, who is much too enthusiastic and perhaps led by the vagaries of youth but nonetheless quite astute, keeps harping on to me about this theory of hers that a smuggling ring is kidnapping Hai and transporting them into human space."`

> 			`	"Of course, a thorough investigation has occurred on our side, and no evidence of such things has turned up, but..."`

Reviewer 10: How can a thorough investigation turn up no evidence of people disappearing???
How deep does this smuggling ring go that it can remove people with such discretion?

(This seems a lot more like the kidnapping story I was promised.)

		add fleet "Anti-Pirate Hai (No Wormhole Travel)" 700

mission "Trader Directions to Hai Space"
	landing
	name "Investigate the Hai"
	description "Follow the coordinates given to you by Captain Casey and meet the Hai."
	source Sunracer
	waypoint "Ultima Thule"
	destination "Allhome"
	to offer
		or
			has "ZZ Event Hai Secret Leaks: RS: offered"
			has "ZZ Event Hai Secret Leaks: FW offered"
	on offer
		conversation
			`Arriving at the only place you can think of where you feel sure you can find out more information, you find that here, like most places, is abuzz with the recent news. Avoiding the many large, cheap, and boisterous establishments that service the many crews and workers of the Syndicate corporate machinery here on Sunracer, you make your way to one of the marginally more up-market places, cunningly marketed as a merchant-only dive.`
			`	The endless cycle of both the news and government propaganda being displayed on most screens in major thoroughfares is remarkably absent here, with only one of the screens on display covering the "violent alien race" at all - running stats on how many different versions of the story are now being told.`

> 			`	Casting around the room you pick out a merchant who looks like they've been around the block a few times. In addition to being a woman who looks like she could pick up most people and break them over her knee, she's conspicuous by the threatening glare you watch her give a younger captain who left a table talking about the Hai to approach her... briefly. As you make your way over though you see her glare falter in a flash of recognition, before subsiding to a reserved scowl.`

Reviewer 3: ```suggestion
			`	Casting around the room, you pick out a merchant who looks like they've been around the block a few times. In addition to looking like she could pick up most people and break them over her knee, she's conspicuous by the threatening glare you watch her give a younger captain who left a table talking about the Hai to approach her... briefly; though as you make your way over, you see her glare falter in a flash of recognition before subsiding to a reserved scowl.`
```
What's happening here? Does the younger captain approach her and is sent away almost immediately? Not clear on that.

Author: Yeah they shouldn't be the same sentence cause they're two distinct events. You watch (from a distance) an exchange happen between her and the younger captain who is swiftly sent away. That moment resolves fully. Then you approach.

Reviewer 3: ```suggestion
			`	Casting around the room, you pick out a merchant who looks like they've been around the block a few times. In addition to looking like she could pick up most people and break them over her knee, she's conspicuous by the threatening glare you watch her give a younger captain who left a table talking about the Hai to approach her... briefly. After the captain is briskly sent away, you make you make your way over. Her glare falters in a flash of recognition before subsiding to a reserved scowl.`
```
How bout this?

Author: Oh now yes that does read well. The double of "you make" looks to be accidental but this structure is good.

> 			`	She waits until you're directly next to her table. "The famous Captain <last>. Which ancient Sumerian deity did I offend to be granted the gift of your attention hey?"`

Reviewer 12: ```suggestion
			`	She waits until you're directly next to her table. "The famous Captain <last>. Which ancient Sumerian deity did I offend to be granted the gift of your attention today?"`
```

Reviewer 12: I was under the impression that this was a typo, is there something I'm missing here?

Author: From Wiki -> "Eh" can also be added to the end of a declarative sentence to turn it into a question seeking agreement. For example: "The weather is nice." becomes "The weather is nice, eh?". This same phrase could also be taken as "The weather is nice, don't you agree?". In this usage, it is similar to [Scots](https://en.wikipedia.org/wiki/Scots_language) "eh?", [English](https://en.wikipedia.org/wiki/English_language) "innit?", [Portuguese](https://en.wikipedia.org/wiki/Portuguese_language) "né?", [Dutch](https://en.wikipedia.org/wiki/Dutch_language) "hè?", [Japanese](https://en.wikipedia.org/wiki/Japanese_language) "ne?", [Mandarin](https://en.wikipedia.org/wiki/Standard_Chinese) "bā" or [French](https://en.wikipedia.org/wiki/French_language) "hein?" (which differs from the French usage of "n'est-ce pas?" ("Is it not?") in that it does not use a (technically double or emphatic) negative).

The equivalent in [South African English](https://en.wikipedia.org/wiki/South_African_English) and most of Southern and Western Australia is "hey".

In oceanic dialects of English (Aus, NZ) both "eh" and "hey" can further be added to the end of an actual question, which is typically otherwise ridiculous to ask, to indicate that the question is rhetorical, subsequently re-inferring the content of the question as a declarative statement. 

In this case it infers the direct statement, "I must have offended an ancient Sumerian deity to be receiving the misfortune of your attention."
-------------------------------------

Until roughly 20 minutes ago I did not realise that this phraseology was as regionally unique as it turns out to be...

Reviewer 12: Huh, I had no idea. My bad. :P

			`	The emphasis she places on the word "gift" is dripping in sarcasm, but her overall tone is wary rather than hostile, and she gestures without comment to the seat opposite for you to sit.`
			choice
				`	"I'm just here to find out more about the Hai."`
					goto innocent
				`	"You looked to be the most experienced person in the room."`
					goto practical

> 				`	"Why ask a rookie, when there's a professional available?"`

Reviewer 3: ```suggestion
				`	"Why ask a rookie when there's a professional available?"`
```

> 			`	"Pff," she scoffs, but despite rolling her eyes a slight smirk undermines her otherwise wary expression. "Flattery from a hero. You've been hanging out with too many politicians <last>."`

Reviewer 3: ```suggestion
			`	She scoffs, but despite her rolling eyes, a slight smirk undermines her otherwise wary expression. "Flattery from a hero. You've been hanging out with too many politicians, <last>."`
```

Author: Repositioning the word rolling feels very off to me.

> 			`	The deliberate omission of your title does not go unnoticed, but she continues. "Still, can't fault your logic, and I suppose I probably owe you one on account of me having got myself stuck on one of those Pug occupied worlds while it was all going on. You're going to ask me about the Hai aren't ya?"`

Reviewer 3: ```suggestion
			`	The deliberate omission of your title does not go unnoticed, but she continues. "Still, can't fault your logic, and I suppose I probably owe you one on account of me having got myself stuck on one of those Pug occupied worlds while it was all going on. You're going to ask me about the Hai, aren't you?"`
```
Feels out of place when she's previously using `you` and `your`.

			choice
				`	"Yes."`
					goto continue
			label innocent

> 			`	"Hah!" she booms, one deep short bark of a laugh. "You and everyone else <last>, and don't go trying to sell me the innocent question line. You didn't single me out cause I was pretty."`

Reviewer 3: ```suggestion
			`	"Hah!" she booms, one deep short bark of a laugh. "You and everyone else, <last>, and don't go trying to sell me the innocent question line. You didn't single me out cause I was pretty."`
```

> 			`	She squints at you and leans back in her seat slightly. "That said, if you're asking me and not any of your new "friends" in the government... maybe you're alright in my books after all."`

Reviewer 3: ```suggestion
			`	She squints at you and leans back in her seat slightly. "That said, if you're asking me and not any of your new 'friends' in the government... maybe you're alright in my books after all."`
```

				goto continue
			label practical

> 			`	Her eyes narrow, but she dips her head in a very small motion of assent, "Well, you're not stupid, I'll give you that. You're trying to find out what most other people are aren't you? You want to know about the Hai."`

Reviewer 3: ```suggestion
			`	Her eyes narrow, but she dips her head in a very small motion of assent. "Well, you're not stupid, I'll give you that. You're trying to find out what most other people are, aren't you? You want to know about the Hai."`
```

			choice
				`	"Yes."`
			label continue

> 			`	She heaves a sigh and lowers her voice until you're forced to lean in. "You can call me Captain Casey, and I will tell you what you want, but then you can leave alright? I like my solitude.`

Reviewer 3: ```suggestion
			`	She heaves a sigh and lowers her voice until you're forced to lean in. "You can call me Captain Casey, and I will tell you what you want, but then you can leave, alright? I like my solitude.`
```

> 			`	"Firstly, I shouldn't be telling you this. Generally speaking they don't like to draw attention to themselves. Secondly though, forget the fearmongering. I make trips up to their space every so often and there aren't a nicer bunch of people anywhere, even if they are a bit furry. The Hai are peaceful, and welcoming of outsiders. They've got themselves a bit of an anarchist faction, that's the footage you've seen, but they ain't no danger to humanity. You should see them for yourself."`

Reviewer 3: ```suggestion
			`	"First, I shouldn't be telling you this. Generally speaking they don't like to draw attention to themselves. Second though, forget the fearmongering. I make trips up to their space every so often, and there aren't a nicer bunch of people anywhere, even if they are a bit furry. The Hai are peaceful and welcoming of outsiders. They've got themselves a bit of an anarchist faction, that's the footage you've seen, but they ain't no danger to humanity. You should see them for yourself."`
```

Author: Can you explain the logic here? I don't understand the reasoning behind any of these changes.

			`	She pulls out her data pad and flicks some coordinates over to you. "Follow those and you'll get there. Keep an eye out for the wormhole."`
			choice
				`	"Thank you."`
					goto wrap
				`	"The wormhole?"`

> 			`	She shakes her head, "See for yourself."`

Reviewer 3: ```suggestion
			`	She shakes her head. "See for yourself."`
```

			label wrap

> 			`	Taking a sip from her mug she nods slightly, once, in your direction. "Intentionally or not, you did me and a lot of other merchants a solid in that war Captain. I respect that, but I'll thank you to leave me in peace if you see me again."`

Reviewer 3: ```suggestion
			`	Taking a sip from her mug she nods slightly, once, in your direction. "Intentionally or not, you did me and a lot of other merchants a solid in that war, Captain. I respect that, but I'll thank you to leave me in peace if you see me again."`
```

			`	Mr. Yang pulls out a datapad. "Well then, I'm afraid I may have some bad news. May I?"`
			`	He gestures to a display and then takes control when indicated. "One of the things I brought with me was Republic Intelligence's estimate of the effectiveness of various information containment measures."`
			`	A series of charts are thrown up on the screen. You're not an expert at data analysis, but highlighted percentages ranging from 4% to 39% do not look promising. The final graphic is a map of stages of what is labeled as 'artifact penetration'. "Artifact penetration is the point at which a government cannot reasonably attempt to pretend something is other than the truth, at least not while still holding to the claim of being a representative government," explains Mr. Yang.`
			`	He checks the date. "Our most optimistic estimates suggest the Republic will be forced to acknowledge the existence of the Hai within three months, or we would lose so much credibility as to compromise electoral stability. The actual time limit is probably sooner."`
			choice
				`	"Would that be so bad?"`

> 			`	"It is always bad to be forced into acknowledging a secret. If we act sooner, we may control the manner of that secret's release," says Yang. "Plus, while the Hai have let the trickle of humans that find the wormhole settle here peacefully, a massive influx of humans is a problem nobody really wants to have to deal with."`

Reviewer 3: ```suggestion
			`	"It is always bad to be forced into acknowledging a secret. If we act sooner, we may control the manner of that secret's release," says Yang. "Plus, while the Hai have let the trickle of humans that find the wormhole settle here peacefully, a massive influx is a problem nobody really wants to have to deal with."`
```
